--- ./discord_94011/src/main/java/com/discord/stores/utilities/RestCallStateKt$logNetworkAction$2.java	2021-12-24 01:10:13.096025300 +0400
+++ ./bluecord/src/main/java/com/discord/stores/utilities/RestCallStateKt$logNetworkAction$2.java	2021-12-24 01:08:33.621260900 +0400
@@ -33,7 +33,7 @@ public final class RestCallStateKt$logNe
         m.checkNotNullExpressionValue(response, "response");
         if (response.a()) {
             Request request = RestCallStateKt.getRequest(response);
-            if (GrowthTeamFeatures.INSTANCE.isNetworkActionLoggingEnabled() && (schema = RestCallStateKt.getSchema(this.$networkActionProvider, new TrackNetworkMetadata(request.b.l, request.f2953c, Long.valueOf((long) response.a.l), null, null, 24), response.b)) != null) {
+            if (GrowthTeamFeatures.INSTANCE.isNetworkActionLoggingEnabled() && (schema = RestCallStateKt.getSchema(this.$networkActionProvider, new TrackNetworkMetadata(request.b.l, request.f2955c, Long.valueOf((long) response.a.l), null, null, 24), response.b)) != null) {
                 this.$tracker.track(schema);
                 return;
             }
