--- ./discord_94011/src/main/java/com/discord/gateway/GatewaySocket.java	2021-12-24 01:10:12.574024500 +0400
+++ ./bluecord/src/main/java/com/discord/gateway/GatewaySocket.java	2021-12-24 01:08:33.070262500 +0400
@@ -425,27 +425,11 @@ public final class GatewaySocket {
         });
     }
 
-    public static final /* synthetic */ void access$cleanup(GatewaySocket gatewaySocket, Function1 function1) {
-        gatewaySocket.cleanup(function1);
-    }
-
-    public static final /* synthetic */ int access$getConnectionState$p(GatewaySocket gatewaySocket) {
-        return gatewaySocket.connectionState;
-    }
-
     public static final /* synthetic */ void access$handleWebSocketOpened(GatewaySocket gatewaySocket, String str) {
         gatewaySocket.handleWebSocketOpened(str);
     }
 
-    public static final /* synthetic */ void access$reset(GatewaySocket gatewaySocket, boolean z2, int i, String str) {
-        gatewaySocket.reset(z2, i, str);
-    }
-
-    public static final /* synthetic */ void access$setConnectionState$p(GatewaySocket gatewaySocket, int i) {
-        gatewaySocket.connectionState = i;
-    }
-
-    private final void cleanup(Function1<? super WebSocket, Unit> function1) {
+    public final void cleanup(Function1<? super WebSocket, Unit> function1) {
         stopHeartbeater();
         clearHelloTimeout();
         if (function1 != null) {
@@ -880,7 +864,7 @@ public final class GatewaySocket {
         gatewaySocket.requestGuildMembers(list, str, list2, num);
     }
 
-    private final void reset(boolean z2, int i, String str) {
+    public final void reset(boolean z2, int i, String str) {
         boolean z3 = false;
         if (i != 4000) {
             this.sessionId = null;
