--- ./discord_94011/src/main/java/com/discord/widgets/channels/list/WidgetChannelsListItemChannelActions.java	2021-12-24 01:10:13.784026100 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/channels/list/WidgetChannelsListItemChannelActions.java	2021-12-24 01:08:34.169263800 +0400
@@ -31,6 +31,7 @@ import kotlin.jvm.functions.Function0;
 import kotlin.jvm.functions.Function1;
 import kotlin.jvm.internal.DefaultConstructorMarker;
 import kotlin.reflect.KProperty;
+import mods.view.ChatPins;
 import rx.Observable;
 import rx.subscriptions.CompositeSubscription;
 
@@ -314,6 +315,7 @@ public final class WidgetChannelsListIte
         }
         view3.setVisibility(i);
         setOnClickAndDismissListener(view3, new WidgetChannelsListItemChannelActions$configureUI$$inlined$apply$lambda$8(this, model));
+        ChatPins.configureUI(this, getBinding().getRoot(), model.getChannel());
     }
 
     private final WidgetChannelsListItemActionsBinding getBinding() {
