--- ./discord_94011/src/main/java/com/discord/widgets/channels/list/WidgetChannelListModel$Companion$getPrivateChannelList$1.java	2021-12-24 01:10:13.760027100 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/channels/list/WidgetChannelListModel$Companion$getPrivateChannelList$1.java	2021-12-24 01:08:34.145261900 +0400
@@ -33,6 +33,7 @@ import java.util.ListIterator;
 import java.util.Map;
 import kotlin.Metadata;
 import kotlin.Unit;
+import mods.view.ChatPins;
 
 @Metadata(bv = {1, 0, 3}, d1 = {"\u0000\u0010\n\u0002\u0010\u0002\n\u0002\b\u0002\n\u0002\u0018\u0002\n\u0002\b\u0003\u0010\u0006\u001a\n \u0001*\u0004\u0018\u00010\u00030\u00032\u000e\u0010\u0002\u001a\n \u0001*\u0004\u0018\u00010\u00000\u0000H\n¢\u0006\u0004\b\u0004\u0010\u0005"}, d2 = {"", "kotlin.jvm.PlatformType", "it", "Lcom/discord/widgets/channels/list/WidgetChannelListModel;", "call", "(Lkotlin/Unit;)Lcom/discord/widgets/channels/list/WidgetChannelListModel;", "<anonymous>"}, k = 3, mv = {1, 4, 2})
 /* compiled from: WidgetChannelListModel.kt */
@@ -77,9 +78,9 @@ public final class WidgetChannelListMode
         }
         Experiment userExperiment = this.$storeExperiments.getUserExperiment("2021-04_contact_sync_android_main", true);
         List<ModelConnectedAccount> connectedAccounts = this.$storeConnections.getConnectedAccounts();
-        List sortedWith = u.sortedWith(channelsForGuild.values(), channelUtils$getSortByMostRecent$1);
-        ArrayList arrayList = new ArrayList(o.collectionSizeOrDefault(sortedWith, 10));
-        Iterator<T> it = sortedWith.iterator();
+        List<Channel> sort = ChatPins.sort(u.sortedWith(channelsForGuild.values(), channelUtils$getSortByMostRecent$1), channelUtils$getSortByMostRecent$1);
+        ArrayList arrayList = new ArrayList(o.collectionSizeOrDefault(sort, 10));
+        Iterator<T> it = sort.iterator();
         while (true) {
             obj = null;
             if (!it.hasNext()) {
