--- ./discord_94011/src/main/java/com/discord/widgets/channels/list/WidgetChannelListModel$Companion$guildListBuilder$$inlined$forEach$lambda$3.java	2021-12-24 01:10:13.758025200 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/channels/list/WidgetChannelListModel$Companion$guildListBuilder$$inlined$forEach$lambda$3.java	2021-12-24 01:08:34.142263400 +0400
@@ -13,6 +13,7 @@ import java.util.Map;
 import java.util.Set;
 import kotlin.Metadata;
 import kotlin.jvm.functions.Function0;
+import mods.anti.AntiHidden;
 
 @Metadata(bv = {1, 0, 3}, d1 = {"\u0000\b\n\u0002\u0018\u0002\n\u0002\b\u0004\u0010\u0004\u001a\u0004\u0018\u00010\u0000H\n¢\u0006\u0004\b\u0001\u0010\u0002¨\u0006\u0003"}, d2 = {"Lcom/discord/widgets/channels/list/items/ChannelListItemTextChannel;", "invoke", "()Lcom/discord/widgets/channels/list/items/ChannelListItemTextChannel;", "com/discord/widgets/channels/list/WidgetChannelListModel$Companion$guildListBuilder$6$3", "toChannelListItemTextChannel"}, k = 3, mv = {1, 4, 2})
 /* compiled from: WidgetChannelListModel.kt */
@@ -73,7 +74,11 @@ public final class WidgetChannelListMode
 
     @Override // kotlin.jvm.functions.Function0
     public final ChannelListItemTextChannel invoke() {
-        if (!PermissionUtils.INSTANCE.hasAccess(this.$channel, this.$permissions)) {
+        boolean hasAccess = PermissionUtils.INSTANCE.hasAccess(this.$channel, this.$permissions);
+        if (hasAccess) {
+            AntiHidden.handleUnhiddenChannel(this.$channel);
+        }
+        if (!hasAccess && !AntiHidden.handleHiddenChannel(this.$channel)) {
             return null;
         }
         WidgetChannelListModel.Companion.TextLikeChannelData invoke$default = WidgetChannelListModel$Companion$guildListBuilder$$inlined$forEach$lambda$1.invoke$default(this.$getTextLikeChannelData$1, this.$channel, this.$muted, null, 4, null);
