--- ./discord_94011/src/main/java/com/discord/widgets/guilds/list/WidgetGuildListAdapter$onCreateViewHolder$7.java	2021-12-24 01:10:14.503026600 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/guilds/list/WidgetGuildListAdapter$onCreateViewHolder$7.java	2021-12-24 01:08:35.097263200 +0400
@@ -32,9 +32,9 @@ public final class WidgetGuildListAdapte
 
     public final void invoke(GuildListItem.PrivateChannelItem privateChannelItem) {
         m.checkNotNullParameter(privateChannelItem, "item");
-        WidgetGuildListAdapter.InteractionListener interactionListener = this.this$0.interactionListener;
+        WidgetGuildListAdapter.InteractionListener access$getInteractionListener$p = WidgetGuildListAdapter.access$getInteractionListener$p(this.this$0);
         View view = this.$itemView;
         m.checkNotNullExpressionValue(view, "itemView");
-        interactionListener.onItemLongPressed(view, privateChannelItem);
+        access$getInteractionListener$p.onItemLongPressed(view, privateChannelItem);
     }
 }
