--- ./discord_94011/src/main/java/i0/e0/a/b.java	2021-12-24 01:10:16.500028000 +0400
+++ ./bluecord/src/main/java/i0/e0/a/b.java	2021-12-24 01:08:36.873263400 +0400
@@ -22,7 +22,7 @@ public final class b<T> implements h<T,
     public final TypeAdapter<T> d;
 
     static {
-        MediaType.a aVar = MediaType.f2950c;
+        MediaType.a aVar = MediaType.f2952c;
     }
 
     public b(Gson gson, TypeAdapter<T> typeAdapter) {
