--- ./discord_94011/src/main/java/f0/e0/e/a.java	2021-12-24 01:10:16.423027600 +0400
+++ ./bluecord/src/main/java/f0/e0/e/a.java	2021-12-24 01:08:36.795261600 +0400
@@ -42,7 +42,7 @@ public final class a implements Intercep
             Response response2 = response.p;
             Response response3 = response.q;
             Response response4 = response.r;
-            long j = response.f2956s;
+            long j = response.f2958s;
             long j2 = response.t;
             c cVar = response.u;
             if (!(i >= 0)) {
@@ -95,7 +95,7 @@ public final class a implements Intercep
             Response.a aVar = new Response.a();
             aVar.g(gVar.f);
             aVar.f(y.HTTP_1_1);
-            aVar.f2957c = 504;
+            aVar.f2959c = 504;
             aVar.e("Unsatisfiable Request (only-if-cached)");
             aVar.g = f0.e0.c.f2800c;
             aVar.k = -1;
@@ -168,7 +168,7 @@ public final class a implements Intercep
                     Object[] array = arrayList.toArray(new String[0]);
                     if (array != null) {
                         aVar3.d(new Headers((String[]) array, null));
-                        aVar3.k = a3.f2956s;
+                        aVar3.k = a3.f2958s;
                         aVar3.l = a3.t;
                         C0303a aVar5 = b;
                         aVar3.b(C0303a.a(aVar5, response));
