--- ./discord_94011/src/main/java/com/discord/widgets/settings/premium/WidgetSettingsPremium$configureButtons$$inlined$forEach$lambda$1.java	2021-12-24 01:10:15.230026300 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/settings/premium/WidgetSettingsPremium$configureButtons$$inlined$forEach$lambda$1.java	2021-12-24 01:08:35.716260200 +0400
@@ -1,8 +1,8 @@
 package com.discord.widgets.settings.premium;
 
 import android.view.View;
-import com.discord.utilities.premium.PremiumUtils;
 import kotlin.Metadata;
+import mods.utils.Alerts;
 
 @Metadata(bv = {1, 0, 3}, d1 = {"\u0000\u0010\n\u0002\u0018\u0002\n\u0002\b\u0002\n\u0002\u0010\u0002\n\u0002\b\u0004\u0010\u0007\u001a\u00020\u00032\u000e\u0010\u0002\u001a\n \u0001*\u0004\u0018\u00010\u00000\u0000H\n¢\u0006\u0004\b\u0004\u0010\u0005¨\u0006\u0006"}, d2 = {"Landroid/view/View;", "kotlin.jvm.PlatformType", "it", "", "onClick", "(Landroid/view/View;)V", "com/discord/widgets/settings/premium/WidgetSettingsPremium$configureButtons$1$1", "<anonymous>"}, k = 3, mv = {1, 4, 2})
 /* compiled from: WidgetSettingsPremium.kt */
@@ -14,6 +14,6 @@ public final class WidgetSettingsPremium
     }
 
     public final void onClick(View view) {
-        PremiumUtils.INSTANCE.openAppleBilling(this.this$0.requireContext());
+        Alerts.alertNitroClick(this.this$0.requireContext());
     }
 }
