--- ./discord_94011/src/main/java/com/discord/widgets/user/usersheet/WidgetUserSheet.java	2021-12-24 01:10:15.577024000 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/user/usersheet/WidgetUserSheet.java	2021-12-24 01:08:35.988260900 +0400
@@ -76,6 +76,8 @@ import kotlin.jvm.functions.Function0;
 import kotlin.jvm.functions.Function1;
 import kotlin.jvm.internal.DefaultConstructorMarker;
 import kotlin.reflect.KProperty;
+import mods.anti.AntiButtons;
+import mods.view.SheetConfig;
 import rx.subscriptions.CompositeSubscription;
 
 public final class WidgetUserSheet extends AppBottomSheet {
@@ -235,6 +237,7 @@ public final class WidgetUserSheet exten
         TextView textView = getBinding().l;
         int i = 8;
         textView.setVisibility(StoreStream.Companion.getUserSettings().getIsDeveloperMode() ? 0 : 8);
+        SheetConfig.configureUserSheet(getView(), loaded.getUser(), getBinding().E);
         textView.setOnClickListener(new WidgetUserSheet$configureDeveloperSection$$inlined$apply$lambda$1(textView, this, loaded));
         TextView textView2 = getBinding().l;
         m.checkNotNullExpressionValue(textView2, "binding.userSheetCopyId");
@@ -415,6 +418,7 @@ public final class WidgetUserSheet exten
                     button8.setVisibility(i);
                     getBinding().f.setOnClickListener(new WidgetUserSheet$configureProfileActionButtons$6(this, loaded));
                     getBinding().f2088z.setOnClickListener(new WidgetUserSheet$configureProfileActionButtons$7(this, userRelationshipType, loaded));
+                    AntiButtons.hideCallButton(getBinding());
                     return;
                 }
                 return;
