--- ./discord_94011/src/main/java/com/discord/widgets/guilds/leave/LeaveGuildDialogViewModel$leaveGuild$4.java	2021-12-24 01:10:14.487024400 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/guilds/leave/LeaveGuildDialogViewModel$leaveGuild$4.java	2021-12-24 01:08:35.087263100 +0400
@@ -28,6 +28,6 @@ public final class LeaveGuildDialogViewM
 
     public final void invoke(Error error) {
         m.checkNotNullParameter(error, "it");
-        LeaveGuildDialogViewModel.access$onLeaveFailed(this.this$0);
+        this.this$0.onLeaveFailed();
     }
 }
