--- ./discord_94011/src/main/java/s/a/z1/b.java	2021-12-24 01:10:16.752025500 +0400
+++ ./bluecord/src/main/java/s/a/z1/b.java	2021-12-24 01:08:37.137262600 +0400
@@ -8,6 +8,6 @@ public final class b {
     public static final t b = new t("OFFER_SUCCESS");
 
     /* renamed from: c  reason: collision with root package name */
-    public static final t f2976c = new t("OFFER_FAILED");
+    public static final t f2978c = new t("OFFER_FAILED");
     public static final t d = new t("POLL_FAILED");
 }
