--- ./discord_94011/src/main/java/s/a/z1/g.java	2021-12-24 01:10:16.754026900 +0400
+++ ./bluecord/src/main/java/s/a/z1/g.java	2021-12-24 01:08:37.139261900 +0400
@@ -12,7 +12,7 @@ import s.a.a.t;
 public class g<E> extends a<E> {
 
     /* renamed from: c  reason: collision with root package name */
-    public final ReentrantLock f2977c = new ReentrantLock();
+    public final ReentrantLock f2979c = new ReentrantLock();
     public Object d = b.a;
 
     public g(Function1<? super E, Unit> function1) {
@@ -31,7 +31,7 @@ public class g<E> extends a<E> {
     public Object e(E e) {
         Function1<E, Unit> function1;
         j<E> j;
-        ReentrantLock reentrantLock = this.f2977c;
+        ReentrantLock reentrantLock = this.f2979c;
         reentrantLock.lock();
         try {
             f<?> b = b();
@@ -75,7 +75,7 @@ public class g<E> extends a<E> {
 
     @Override // s.a.z1.a
     public boolean f(h<? super E> hVar) {
-        ReentrantLock reentrantLock = this.f2977c;
+        ReentrantLock reentrantLock = this.f2979c;
         reentrantLock.lock();
         try {
             return super.f(hVar);
@@ -96,7 +96,7 @@ public class g<E> extends a<E> {
 
     @Override // s.a.z1.a
     public Object i() {
-        ReentrantLock reentrantLock = this.f2977c;
+        ReentrantLock reentrantLock = this.f2979c;
         reentrantLock.lock();
         try {
             Object obj = this.d;
