--- ./discord_94011/src/main/java/s/a/g1.java	2021-12-24 01:10:16.734025100 +0400
+++ ./bluecord/src/main/java/s/a/g1.java	2021-12-24 01:08:37.119265200 +0400
@@ -411,7 +411,7 @@ public class g1 implements Job, q, o1 {
                 }
                 throw new IllegalStateException(str, th);
             }
-        } while (V == h1.f2973c);
+        } while (V == h1.f2975c);
         return V;
     }
 
@@ -594,12 +594,12 @@ public class g1 implements Job, q, o1 {
             if (z2) {
                 return obj2;
             }
-            return h1.f2973c;
+            return h1.f2975c;
         }
         y0 y0Var2 = (y0) obj;
         l1 E = E(y0Var2);
         if (E == null) {
-            return h1.f2973c;
+            return h1.f2975c;
         }
         p pVar = null;
         b bVar = (b) (!(y0Var2 instanceof b) ? null : y0Var2);
@@ -622,7 +622,7 @@ public class g1 implements Job, q, o1 {
                     th = null;
                 }
             } else {
-                return h1.f2973c;
+                return h1.f2975c;
             }
         }
     }
@@ -774,7 +774,7 @@ public class g1 implements Job, q, o1 {
                 Object F = F();
                 if ((F instanceof y0) && (!(F instanceof b) || ((b) F)._isCompleting == 0)) {
                     tVar2 = V(F, new v(v(obj), false, 2));
-                    if (tVar2 != h1.f2973c) {
+                    if (tVar2 != h1.f2975c) {
                         break;
                     }
                 } else {
@@ -817,7 +817,7 @@ public class g1 implements Job, q, o1 {
                     } else {
                         Object V = V(F2, new v(th2, false, 2));
                         if (V != h1.a) {
-                            if (V != h1.f2973c) {
+                            if (V != h1.f2975c) {
                                 tVar2 = V;
                                 break;
                             }
