--- ./discord_94011/src/main/java/s/a/k.java	2021-12-24 01:10:16.737024200 +0400
+++ ./bluecord/src/main/java/s/a/k.java	2021-12-24 01:08:37.122263500 +0400
@@ -126,7 +126,7 @@ public class k<T> extends i0<T> implemen
                             if (iVar != null) {
                                 p(iVar, th);
                             }
-                            Function1<Throwable, Unit> function1 = uVar.f2975c;
+                            Function1<Throwable, Unit> function1 = uVar.f2977c;
                             if (function1 != null) {
                                 q(function1, th);
                                 return;
@@ -460,7 +460,7 @@ public class k<T> extends i0<T> implemen
                 if (obj2 instanceof m) {
                     m mVar = (m) obj2;
                     Objects.requireNonNull(mVar);
-                    if (m.f2974c.compareAndSet(mVar, 0, 1)) {
+                    if (m.f2976c.compareAndSet(mVar, 0, 1)) {
                         if (function1 != null) {
                             q(function1, mVar.b);
                             return;
