--- ./discord_94011/src/main/java/androidx/core/graphics/TypefaceCompatApi21Impl.java	2021-12-24 01:10:08.940030200 +0400
+++ ./bluecord/src/main/java/androidx/core/graphics/TypefaceCompatApi21Impl.java	2021-12-24 01:08:29.711265500 +0400
@@ -29,11 +29,11 @@ public class TypefaceCompatApi21Impl ext
     private static final String CREATE_FROM_FAMILIES_WITH_DEFAULT_METHOD = "createFromFamiliesWithDefault";
     private static final String FONT_FAMILY_CLASS = "android.graphics.FontFamily";
     private static final String TAG = "TypefaceCompatApi21Impl";
-    private static Method sAddFontWeightStyle = null;
-    private static Method sCreateFromFamiliesWithDefault = null;
-    private static Class<?> sFontFamily = null;
-    private static Constructor<?> sFontFamilyCtor = null;
-    private static boolean sHasInitBeenCalled = false;
+    private static Method sAddFontWeightStyle;
+    private static Method sCreateFromFamiliesWithDefault;
+    private static Class<?> sFontFamily;
+    private static Constructor<?> sFontFamilyCtor;
+    private static boolean sHasInitBeenCalled;
 
     private static boolean addFontWeightStyle(Object obj, String str, int i, boolean z2) {
         init();
