--- ./discord_94011/src/main/java/com/discord/api/channel/Channel.java	2021-12-24 01:10:12.214027200 +0400
+++ ./bluecord/src/main/java/com/discord/api/channel/Channel.java	2021-12-24 01:08:32.761263100 +0400
@@ -33,6 +33,7 @@ public final class Channel {
     private final Integer defaultAutoArchiveDuration;
     private transient GuildHashes guildHashes;
     private final long guildId;
+    private boolean hidden;
     private final String icon;
 
     /* renamed from: id  reason: collision with root package name */
@@ -43,7 +44,7 @@ public final class Channel {
     private final transient List<Long> memberIdsPreview;
     private final String memberListId;
     private transient Integer messageCount;
-    private final String name;
+    private String name;
     private final List<ChannelRecipientNick> nicks;
     private final boolean nsfw;
     private final long originChannelId;
@@ -244,6 +245,10 @@ public final class Channel {
         return this.lastMessageId;
     }
 
+    public boolean isHidden() {
+        return this.hidden;
+    }
+
     public final ThreadMember j() {
         return this.member;
     }
@@ -284,6 +289,14 @@ public final class Channel {
         return this.permissionOverwrites;
     }
 
+    public void setHidden(boolean z2) {
+        this.hidden = z2;
+    }
+
+    public void setName(String str) {
+        this.name = str;
+    }
+
     public final int t() {
         return this.position;
     }
