--- ./discord_94011/src/main/java/com/airbnb/lottie/parser/AnimatableValueParser.java	2021-12-24 01:10:11.642024200 +0400
+++ ./bluecord/src/main/java/com/airbnb/lottie/parser/AnimatableValueParser.java	2021-12-24 01:08:32.114264800 +0400
@@ -144,6 +144,7 @@ import java.util.zip.Inflater;
 import kotlin.Unit;
 import kotlin.jvm.functions.Function0;
 import kotlin.jvm.functions.Function1;
+import mods.ThemingTools;
 import org.checkerframework.checker.nullness.qual.EnsuresNonNull;
 import org.checkerframework.dataflow.qual.Pure;
 import org.xmlpull.v1.XmlPullParser;
@@ -783,11 +784,11 @@ public class AnimatableValueParser {
     }
 
     /* JADX INFO: finally extract failed */
-    /* JADX WARNING: Code restructure failed: missing block: B:142:0x024a, code lost:
-        if (r15 != false) goto L_0x0266;
+    /* JADX WARNING: Code restructure failed: missing block: B:142:0x024c, code lost:
+        if (r15 != false) goto L_0x0268;
      */
-    /* JADX WARNING: Code restructure failed: missing block: B:156:0x0264, code lost:
-        if (r16 != false) goto L_0x0266;
+    /* JADX WARNING: Code restructure failed: missing block: B:156:0x0266, code lost:
+        if (r16 != false) goto L_0x0268;
      */
     public static c.f.g.f.a N2(c.f.g.f.a aVar, Context context, AttributeSet attributeSet) {
         int i;
@@ -1059,6 +1060,7 @@ public class AnimatableValueParser {
             fArr[7] = f4;
             fArr[6] = f4;
         }
+        ThemingTools.showSquarePicsDamnIt(aVar, attributeSet);
         return aVar;
     }
 
