--- ./discord_94011/src/main/java/com/discord/widgets/settings/WidgetSettingsAppearance$Model$Companion$get$1.java	2021-12-24 01:10:15.070023800 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/settings/WidgetSettingsAppearance$Model$Companion$get$1.java	2021-12-24 01:08:35.527260600 +0400
@@ -6,6 +6,7 @@ import com.discord.utilities.user.UserUt
 import com.discord.widgets.settings.WidgetSettingsAppearance;
 import d0.z.d.m;
 import kotlin.Metadata;
+import mods.constants.Constants;
 import rx.functions.Func3;
 
 @Metadata(bv = {1, 0, 3}, d1 = {"\u0000\u001c\n\u0002\u0018\u0002\n\u0002\b\u0002\n\u0002\u0018\u0002\n\u0000\n\u0002\u0010\u000b\n\u0000\n\u0002\u0018\u0002\n\u0002\b\u0003\u0010\n\u001a\n \u0001*\u0004\u0018\u00010\u00070\u00072\u000e\u0010\u0002\u001a\n \u0001*\u0004\u0018\u00010\u00000\u00002\u000e\u0010\u0004\u001a\n \u0001*\u0004\u0018\u00010\u00030\u00032\u000e\u0010\u0006\u001a\n \u0001*\u0004\u0018\u00010\u00050\u0005H\n¢\u0006\u0004\b\b\u0010\t"}, d2 = {"Lcom/discord/models/user/MeUser;", "kotlin.jvm.PlatformType", "meUser", "Lcom/discord/stores/StoreUserSettingsSystem$Settings;", "settings", "", "isPureEvilVisible", "Lcom/discord/widgets/settings/WidgetSettingsAppearance$Model;", "call", "(Lcom/discord/models/user/MeUser;Lcom/discord/stores/StoreUserSettingsSystem$Settings;Ljava/lang/Boolean;)Lcom/discord/widgets/settings/WidgetSettingsAppearance$Model;", "<anonymous>"}, k = 3, mv = {1, 4, 2})
@@ -14,22 +15,21 @@ public final class WidgetSettingsAppeara
     public static final WidgetSettingsAppearance$Model$Companion$get$1 INSTANCE = new WidgetSettingsAppearance$Model$Companion$get$1();
 
     public final WidgetSettingsAppearance.Model call(MeUser meUser, StoreUserSettingsSystem.Settings settings, Boolean bool) {
-        boolean z2;
         String theme = settings.getTheme();
         m.checkNotNullExpressionValue(bool, "isPureEvilVisible");
         if (!bool.booleanValue()) {
             UserUtils userUtils = UserUtils.INSTANCE;
             m.checkNotNullExpressionValue(meUser, "meUser");
             if (!userUtils.isStaff(meUser)) {
-                z2 = false;
+                boolean z2 = Constants.TRUE;
                 int fontScale = settings.getFontScale();
                 m.checkNotNullExpressionValue(meUser, "meUser");
                 return new WidgetSettingsAppearance.Model(theme, z2, fontScale, meUser);
             }
         }
-        z2 = true;
+        boolean z22 = Constants.TRUE;
         int fontScale2 = settings.getFontScale();
         m.checkNotNullExpressionValue(meUser, "meUser");
-        return new WidgetSettingsAppearance.Model(theme, z2, fontScale2, meUser);
+        return new WidgetSettingsAppearance.Model(theme, z22, fontScale2, meUser);
     }
 }
