--- ./discord_94011/src/main/java/androidx/core/view/KeyEventDispatcher.java	2021-12-24 01:10:09.014025100 +0400
+++ ./bluecord/src/main/java/androidx/core/view/KeyEventDispatcher.java	2021-12-24 01:08:29.818262800 +0400
@@ -17,9 +17,9 @@ import java.lang.reflect.Method;
 
 @RestrictTo({RestrictTo.Scope.LIBRARY_GROUP_PREFIX})
 public class KeyEventDispatcher {
-    private static boolean sActionBarFieldsFetched = false;
-    private static Method sActionBarOnMenuKeyMethod = null;
-    private static boolean sDialogFieldsFetched = false;
+    private static boolean sActionBarFieldsFetched;
+    private static Method sActionBarOnMenuKeyMethod;
+    private static boolean sDialogFieldsFetched;
     private static Field sDialogKeyListenerField;
 
     public interface Component {
