--- ./discord_94011/src/main/java/androidx/core/view/WindowInsetsCompat.java	2021-12-24 01:10:09.045024900 +0400
+++ ./bluecord/src/main/java/androidx/core/view/WindowInsetsCompat.java	2021-12-24 01:08:29.863264100 +0400
@@ -314,12 +314,12 @@ public class WindowInsetsCompat {
 
     @RequiresApi(20)
     public static class Impl20 extends Impl {
-        private static Class<?> sAttachInfoClass = null;
-        private static Field sAttachInfoField = null;
-        private static Method sGetViewRootImplMethod = null;
-        private static Class<?> sViewRootImplClass = null;
-        private static Field sVisibleInsetsField = null;
-        private static boolean sVisibleRectReflectionFetched = false;
+        private static Class<?> sAttachInfoClass;
+        private static Field sAttachInfoField;
+        private static Method sGetViewRootImplMethod;
+        private static Class<?> sViewRootImplClass;
+        private static Field sVisibleInsetsField;
+        private static boolean sVisibleRectReflectionFetched;
         private Insets[] mOverriddenInsets;
         @NonNull
         public final WindowInsets mPlatformInsets;
@@ -1014,10 +1014,10 @@ public class WindowInsetsCompat {
 
     @RequiresApi(api = 20)
     public static class BuilderImpl20 extends BuilderImpl {
-        private static Constructor<WindowInsets> sConstructor = null;
-        private static boolean sConstructorFetched = false;
-        private static Field sConsumedField = null;
-        private static boolean sConsumedFieldFetched = false;
+        private static Constructor<WindowInsets> sConstructor;
+        private static boolean sConstructorFetched;
+        private static Field sConsumedField;
+        private static boolean sConsumedFieldFetched;
         private WindowInsets mPlatformInsets;
         private Insets mStableInsets;
 
