--- ./discord_94011/src/main/java/com/discord/widgets/servers/guild_role_subscription/edit_tier/ServerSettingsGuildRoleSubscriptionEditTierViewModel$submitChanges$2.java	2021-12-24 01:10:14.983026900 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/servers/guild_role_subscription/edit_tier/ServerSettingsGuildRoleSubscriptionEditTierViewModel$submitChanges$2.java	2021-12-24 01:08:35.453263000 +0400
@@ -31,7 +31,7 @@ public final class ServerSettingsGuildRo
 
     public final void invoke(Error error) {
         m.checkNotNullParameter(error, "it");
-        this.this$0.updateViewState(ServerSettingsGuildRoleSubscriptionEditTierViewModel.ViewState.Loaded.copy$default(this.$viewState, 0, false, false, false, 7, null));
-        this.this$0.emitEvent(ServerSettingsGuildRoleSubscriptionEditTierViewModel.Event.SubmitFailure.INSTANCE);
+        ServerSettingsGuildRoleSubscriptionEditTierViewModel.access$updateViewState(this.this$0, ServerSettingsGuildRoleSubscriptionEditTierViewModel.ViewState.Loaded.copy$default(this.$viewState, 0, false, false, false, 7, null));
+        ServerSettingsGuildRoleSubscriptionEditTierViewModel.access$emitEvent(this.this$0, ServerSettingsGuildRoleSubscriptionEditTierViewModel.Event.SubmitFailure.INSTANCE);
     }
 }
