--- ./discord_94011/src/main/java/i0/p.java	2021-12-24 01:10:16.515026200 +0400
+++ ./bluecord/src/main/java/i0/p.java	2021-12-24 01:08:36.888263100 +0400
@@ -256,7 +256,7 @@ public final class p<T> implements d<T>
             aVar6.g(wVar);
             Headers c2 = vVar.h.c();
             m.checkParameterIsNotNull(c2, "headers");
-            aVar6.f2954c = c2.e();
+            aVar6.f2956c = c2.e();
             aVar6.c(vVar.f2918c, aVar5);
             aVar6.e(k.class, new k(wVar2.a, arrayList));
             e b2 = aVar.b(aVar6.a());
@@ -361,7 +361,7 @@ public final class p<T> implements d<T>
         Response response2 = response.p;
         Response response3 = response.q;
         Response response4 = response.r;
-        long j2 = response.f2956s;
+        long j2 = response.f2958s;
         long j3 = response.t;
         f0.e0.g.c cVar = response.u;
         c cVar2 = new c(responseBody.b(), responseBody.a());
