--- ./discord_94011/src/main/java/i0/v.java	2021-12-24 01:10:16.516025200 +0400
+++ ./bluecord/src/main/java/i0/v.java	2021-12-24 01:08:36.890265000 +0400
@@ -104,7 +104,7 @@ public final class v {
     public void b(String str, String str2) {
         if ("Content-Type".equalsIgnoreCase(str)) {
             try {
-                MediaType.a aVar = MediaType.f2950c;
+                MediaType.a aVar = MediaType.f2952c;
                 this.i = MediaType.a.a(str2);
             } catch (IllegalArgumentException e2) {
                 throw new IllegalArgumentException(c.d.b.a.a.u("Malformed content type: ", str2), e2);
