--- ./discord_94011/src/main/java/com/discord/utilities/voice/CallSoundManager.java	2021-12-24 01:10:13.566026700 +0400
+++ ./bluecord/src/main/java/com/discord/utilities/voice/CallSoundManager.java	2021-12-24 01:08:33.950265000 +0400
@@ -278,10 +278,6 @@ public final class CallSoundManager {
         this.streamingUserIds = n.emptyList();
     }
 
-    public static final /* synthetic */ AppSoundManager access$getAppSoundManager$p(CallSoundManager callSoundManager) {
-        return callSoundManager.appSoundManager;
-    }
-
     /*  JADX ERROR: IF instruction can be used only in fallback mode
         jadx.core.utils.exceptions.CodegenException: IF instruction can be used only in fallback mode
         	at jadx.core.codegen.InsnGen.fallbackOnlyInsn(InsnGen.java:604)
