--- ./discord_94011/src/main/java/f0/e0/g/a.java	2021-12-24 01:10:16.425024800 +0400
+++ ./bluecord/src/main/java/f0/e0/g/a.java	2021-12-24 01:08:36.797262600 +0400
@@ -38,7 +38,7 @@ public final class a implements Intercep
         m.checkParameterIsNotNull(xVar, "client");
         m.checkParameterIsNotNull(gVar, "chain");
         try {
-            c cVar = new c(eVar, eVar.j, dVar, dVar.a(gVar.g, gVar.h, gVar.i, xVar.L, xVar.q, !m.areEqual(gVar.f.f2953c, "GET")).k(xVar, gVar));
+            c cVar = new c(eVar, eVar.j, dVar, dVar.a(gVar.g, gVar.h, gVar.i, xVar.L, xVar.q, !m.areEqual(gVar.f.f2955c, "GET")).k(xVar, gVar));
             eVar.q = cVar;
             eVar.v = cVar;
             synchronized (eVar) {
