--- ./discord_94011/src/main/java/f0/e0/g/j.java	2021-12-24 01:10:16.428025600 +0400
+++ ./bluecord/src/main/java/f0/e0/g/j.java	2021-12-24 01:08:36.800261700 +0400
@@ -318,7 +318,7 @@ public final class j extends e.c impleme
         Response.a aVar2 = new Response.a();
         aVar2.g(a);
         aVar2.f(y.HTTP_1_1);
-        aVar2.f2957c = 407;
+        aVar2.f2959c = 407;
         aVar2.e("Preemptive Authenticate");
         aVar2.g = c.f2800c;
         aVar2.k = -1;
