--- ./discord_94011/src/main/java/com/discord/widgets/servers/premiumguild/PremiumGuildSubscriptionInProgressViewModel.java	2021-12-24 01:10:15.011025000 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/servers/premiumguild/PremiumGuildSubscriptionInProgressViewModel.java	2021-12-24 01:08:35.477261200 +0400
@@ -305,7 +305,11 @@ public final class PremiumGuildSubscript
         }
     }
 
-    public final void handlePremiumGuildSubscriptionCompleted() {
+    public static final /* synthetic */ void access$handlePremiumGuildSubscriptionCompleted(PremiumGuildSubscriptionInProgressViewModel premiumGuildSubscriptionInProgressViewModel) {
+        premiumGuildSubscriptionInProgressViewModel.handlePremiumGuildSubscriptionCompleted();
+    }
+
+    private final void handlePremiumGuildSubscriptionCompleted() {
         Object obj;
         this.storePremiumGuildSubscription.fetchUserGuildPremiumState();
         ViewState viewState = (ViewState) requireViewState();
