--- ./discord_94011/src/main/java/s/a/a/a.java	2021-12-24 01:10:16.715025700 +0400
+++ ./bluecord/src/main/java/s/a/a/a.java	2021-12-24 01:08:37.098262700 +0400
@@ -13,7 +13,7 @@ public final class a {
     public static final Function2<Object, CoroutineContext.Element, Object> b = b.i;
 
     /* renamed from: c  reason: collision with root package name */
-    public static final Function2<s1<?>, CoroutineContext.Element, s1<?>> f2965c = c.i;
+    public static final Function2<s1<?>, CoroutineContext.Element, s1<?>> f2967c = c.i;
     public static final Function2<x, CoroutineContext.Element, x> d = C0340a.j;
     public static final Function2<x, CoroutineContext.Element, x> e = C0340a.i;
 
@@ -39,7 +39,7 @@ public final class a {
                 x xVar2 = xVar;
                 CoroutineContext.Element element2 = element;
                 if (element2 instanceof s1) {
-                    CoroutineContext coroutineContext = xVar2.f2969c;
+                    CoroutineContext coroutineContext = xVar2.f2971c;
                     Object[] objArr = xVar2.a;
                     int i3 = xVar2.b;
                     xVar2.b = i3 + 1;
@@ -50,7 +50,7 @@ public final class a {
                 x xVar3 = xVar;
                 CoroutineContext.Element element3 = element;
                 if (element3 instanceof s1) {
-                    Object C = ((s1) element3).C(xVar3.f2969c);
+                    Object C = ((s1) element3).C(xVar3.f2971c);
                     Object[] objArr2 = xVar3.a;
                     int i4 = xVar3.b;
                     xVar3.b = i4 + 1;
@@ -118,7 +118,7 @@ public final class a {
                 coroutineContext.fold(obj, e);
                 return;
             }
-            Object fold = coroutineContext.fold(null, f2965c);
+            Object fold = coroutineContext.fold(null, f2967c);
             Objects.requireNonNull(fold, "null cannot be cast to non-null type kotlinx.coroutines.ThreadContextElement<kotlin.Any?>");
             ((s1) fold).x(coroutineContext, obj);
         }
