--- ./discord_94011/src/main/java/com/discord/widgets/guilds/WidgetGuildSelector.java	2021-12-24 01:10:14.400024600 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/guilds/WidgetGuildSelector.java	2021-12-24 01:08:35.008264100 +0400
@@ -40,6 +40,7 @@ import kotlin.jvm.functions.Function1;
 import kotlin.jvm.functions.Function2;
 import kotlin.jvm.internal.DefaultConstructorMarker;
 import kotlin.reflect.KProperty;
+import mods.ThemingTools;
 import rx.Observable;
 import rx.subscriptions.CompositeSubscription;
 
@@ -134,7 +135,7 @@ public final class WidgetGuildSelector e
                 Integer valueOf = Integer.valueOf(ColorCompat.getThemedColor(view, 2130968907));
                 SimpleDraweeView simpleDraweeView2 = this.binding.f1883c;
                 m.checkNotNullExpressionValue(simpleDraweeView2, "binding.itemIcon");
-                SimpleDraweeViewExtensionsKt.setGuildIcon$default(simpleDraweeView, false, guild4, (float) simpleDraweeView2.getResources().getDimensionPixelSize(2131165448), null, valueOf, null, null, true, null, 360, null);
+                SimpleDraweeViewExtensionsKt.setGuildIcon$default(simpleDraweeView, false, guild4, (float) ThemingTools.fixIconRadius(simpleDraweeView2.getResources().getDimensionPixelSize(2131165448)), null, valueOf, null, null, true, null, 360, null);
                 FrameLayout frameLayout = this.binding.b;
                 m.checkNotNullExpressionValue(frameLayout, "binding.itemAvatarWrap");
                 frameLayout.setClipToOutline(true);
