--- ./discord_94011/src/main/java/com/discord/widgets/guilds/WidgetGuildFolderSettings.java	2021-12-24 01:10:14.394026400 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/guilds/WidgetGuildFolderSettings.java	2021-12-24 01:08:35.003262500 +0400
@@ -77,6 +77,10 @@ public final class WidgetGuildFolderSett
         return widgetGuildFolderSettings.argbColorToRGB(i);
     }
 
+    public static final /* synthetic */ WidgetGuildFolderSettingsViewModel access$getViewModel$p(WidgetGuildFolderSettings widgetGuildFolderSettings) {
+        return widgetGuildFolderSettings.getViewModel();
+    }
+
     private final int argbColorToRGB(int i) {
         return Color.argb(0, Color.red(i), Color.green(i), Color.blue(i));
     }
@@ -179,7 +183,7 @@ public final class WidgetGuildFolderSett
         return iArr;
     }
 
-    public final WidgetGuildFolderSettingsViewModel getViewModel() {
+    private final WidgetGuildFolderSettingsViewModel getViewModel() {
         return (WidgetGuildFolderSettingsViewModel) this.viewModel$delegate.getValue();
     }
 
