--- ./discord_94011/src/main/java/okhttp3/internal/publicsuffix/PublicSuffixDatabase.java	2021-12-24 01:10:16.620024400 +0400
+++ ./bluecord/src/main/java/okhttp3/internal/publicsuffix/PublicSuffixDatabase.java	2021-12-24 01:08:37.028263000 +0400
@@ -29,7 +29,7 @@ public final class PublicSuffixDatabase
     public static final List<String> b = m.listOf("*");
 
     /* renamed from: c  reason: collision with root package name */
-    public static final PublicSuffixDatabase f2958c = new PublicSuffixDatabase();
+    public static final PublicSuffixDatabase f2960c = new PublicSuffixDatabase();
     public static final a d = new a(null);
     public final AtomicBoolean e = new AtomicBoolean(false);
     public final CountDownLatch f = new CountDownLatch(1);
