--- ./discord_94011/src/main/java/z/a.java	2021-12-24 01:10:16.768024400 +0400
+++ ./bluecord/src/main/java/z/a.java	2021-12-24 01:08:37.152261600 +0400
@@ -10,7 +10,7 @@ public final class a {
     public static final int b;
 
     /* renamed from: c  reason: collision with root package name */
-    public static final int f2978c;
+    public static final int f2980c;
     public static final int d;
     public final Executor e = new b(null);
 
@@ -27,7 +27,7 @@ public final class a {
     static {
         int availableProcessors = Runtime.getRuntime().availableProcessors();
         b = availableProcessors;
-        f2978c = availableProcessors + 1;
+        f2980c = availableProcessors + 1;
         d = (availableProcessors * 2) + 1;
     }
 }
