--- ./discord_94011/src/main/java/s/a/b2/c.java	2021-12-24 01:10:16.727027600 +0400
+++ ./bluecord/src/main/java/s/a/b2/c.java	2021-12-24 01:08:37.111262300 +0400
@@ -15,7 +15,7 @@ public class c extends ExecutorCoroutine
 
     public c(int i2, int i3, String str, int i4) {
         i2 = (i4 & 1) != 0 ? k.b : i2;
-        i3 = (i4 & 2) != 0 ? k.f2971c : i3;
+        i3 = (i4 & 2) != 0 ? k.f2973c : i3;
         String str2 = (i4 & 4) != 0 ? "DefaultDispatcher" : null;
         long j2 = k.d;
         this.j = i2;
