--- ./discord_94011/src/main/java/s/a/b2/k.java	2021-12-24 01:10:16.729026500 +0400
+++ ./bluecord/src/main/java/s/a/b2/k.java	2021-12-24 01:08:37.114260900 +0400
@@ -11,7 +11,7 @@ public final class k {
     public static final int b;
 
     /* renamed from: c  reason: collision with root package name */
-    public static final int f2971c;
+    public static final int f2973c;
     public static final long d = TimeUnit.SECONDS.toNanos(a.Z("kotlinx.coroutines.scheduler.keep.alive.sec", 60, 0, 0, 12, null));
     public static l e = f.a;
 
@@ -20,6 +20,6 @@ public final class k {
         int i = u.a;
         int Y = a.Y("kotlinx.coroutines.scheduler.core.pool.size", f.coerceAtLeast(i, 2), 1, 0, 8, null);
         b = Y;
-        f2971c = a.Y("kotlinx.coroutines.scheduler.max.pool.size", f.coerceIn(i * 128, Y, 2097150), 0, 2097150, 4, null);
+        f2973c = a.Y("kotlinx.coroutines.scheduler.max.pool.size", f.coerceIn(i * 128, Y, 2097150), 0, 2097150, 4, null);
     }
 }
