--- ./discord_94011/src/main/java/com/discord/widgets/guilds/create/WidgetCreationIntent$configureFriendsButton$1.java	2021-12-24 01:10:14.418025600 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/guilds/create/WidgetCreationIntent$configureFriendsButton$1.java	2021-12-24 01:08:35.023261800 +0400
@@ -13,6 +13,6 @@ public final class WidgetCreationIntent$
     }
 
     public final void onClick(View view) {
-        this.this$0.onSelectionPressed(Boolean.FALSE);
+        WidgetCreationIntent.access$onSelectionPressed(this.this$0, Boolean.FALSE);
     }
 }
