--- ./discord_94011/src/main/java/com/google/android/gms/auth/api/signin/internal/SignInHubActivity.java	2021-12-24 01:10:15.823026300 +0400
+++ ./bluecord/src/main/java/com/google/android/gms/auth/api/signin/internal/SignInHubActivity.java	2021-12-24 01:08:36.254263100 +0400
@@ -24,7 +24,7 @@ import java.util.Set;
 @KeepName
 /* compiled from: com.google.android.gms:play-services-auth@@19.0.0 */
 public class SignInHubActivity extends FragmentActivity {
-    public static boolean i = false;
+    public static boolean i;
     public boolean j = false;
     public SignInConfiguration k;
     public boolean l;
