--- ./discord_94011/src/main/java/com/discord/utilities/error/Error.java	2021-12-24 01:10:13.217023500 +0400
+++ ./bluecord/src/main/java/com/discord/utilities/error/Error.java	2021-12-24 01:08:33.695260600 +0400
@@ -452,7 +452,7 @@ public class Error {
             retrofit2.Response<?> response3 = httpException.i;
             if (response3 != null) {
                 headers = response3.a.n;
-                ResponseBody responseBody = response3.f2964c;
+                ResponseBody responseBody = response3.f2966c;
                 if (responseBody != null) {
                     MediaType b = responseBody.b();
                     str2 = b != null ? b.d : str5;
