--- ./discord_94011/src/main/java/com/discord/stores/StoreEmojiGuild$fetchGuildEmoji$1.java	2021-12-24 01:10:12.783026500 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreEmojiGuild$fetchGuildEmoji$1.java	2021-12-24 01:08:33.278262200 +0400
@@ -31,6 +31,6 @@ public final class StoreEmojiGuild$fetch
 
     public final void invoke(List<ModelEmojiGuild> list) {
         m.checkNotNullParameter(list, "emojis");
-        this.this$0.handleGuildEmojisLoaded(this.$guildId, list);
+        StoreEmojiGuild.access$handleGuildEmojisLoaded(this.this$0, this.$guildId, list);
     }
 }
