--- ./discord_94011/src/main/java/com/discord/stores/StoreEntitlements.java	2021-12-24 01:10:12.784024700 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreEntitlements.java	2021-12-24 01:08:33.279263300 +0400
@@ -148,6 +148,10 @@ public final class StoreEntitlements ext
         this(dispatcher2, observationDeck2, (i & 4) != 0 ? RestAPI.Companion.getApi() : restAPI2);
     }
 
+    public static final /* synthetic */ Dispatcher access$getDispatcher$p(StoreEntitlements storeEntitlements) {
+        return storeEntitlements.dispatcher;
+    }
+
     public final void fetchMyEntitlementsForApplication(long j) {
         this.dispatcher.schedule(new StoreEntitlements$fetchMyEntitlementsForApplication$1(this));
         ObservableExtensionsKt.appSubscribe$default(ObservableExtensionsKt.restSubscribeOn$default(RestAPIInterface.DefaultImpls.getMyEntitlements$default(this.restAPI, j, false, 2, null), false, 1, null), StoreEntitlements.class, (Context) null, (Function1) null, new StoreEntitlements$fetchMyEntitlementsForApplication$2(this), (Function0) null, (Function0) null, new StoreEntitlements$fetchMyEntitlementsForApplication$3(this, j), 54, (Object) null);
