--- ./discord_94011/src/main/java/com/discord/stores/StoreStickers.java	2021-12-24 01:10:12.992026700 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreStickers.java	2021-12-24 01:08:33.510262200 +0400
@@ -142,6 +142,10 @@ public final class StoreStickers extends
         this(dispatcher2, (i & 2) != 0 ? RestAPI.Companion.getApi() : restAPI, (i & 4) != 0 ? ObservationDeckProvider.get() : observationDeck2, (i & 8) != 0 ? ClockFactory.get() : clock2);
     }
 
+    public static final /* synthetic */ List access$getEnabledStickerPacksSnapshot$p(StoreStickers storeStickers) {
+        return storeStickers.enabledStickerPacksSnapshot;
+    }
+
     public final void fetchEnabledStickerDirectory() {
         if (this.lastFetchedEnabledPacks + 300000 < this.clock.currentTimeMillis()) {
             ObservableExtensionsKt.appSubscribe$default(ObservableExtensionsKt.restSubscribeOn$default(this.api.getStickerPacks(), false, 1, null), StoreStickers.class, (Context) null, (Function1) null, (Function1) null, (Function0) null, (Function0) null, new StoreStickers$fetchEnabledStickerDirectory$1(this), 62, (Object) null);
