--- ./discord_94011/src/main/java/com/discord/stores/StoreDirectories$observeDirectoriesForChannel$1.java	2021-12-24 01:10:12.771025700 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreDirectories$observeDirectoriesForChannel$1.java	2021-12-24 01:08:33.268264100 +0400
@@ -23,6 +23,6 @@ public final class StoreDirectories$obse
     /* Return type fixed from 'com.discord.stores.utilities.RestCallState<java.util.List<com.discord.api.directory.DirectoryEntry>>' to match base method */
     @Override // kotlin.jvm.functions.Function0
     public final RestCallState<? extends List<? extends DirectoryEntry>> invoke() {
-        return (RestCallState) this.this$0.directoriesMapSnapshot.get(Long.valueOf(this.$channelId));
+        return (RestCallState) StoreDirectories.access$getDirectoriesMapSnapshot$p(this.this$0).get(Long.valueOf(this.$channelId));
     }
 }
