--- ./discord_94011/src/main/java/com/discord/stores/StoreEmojiGuild$handleGuildEmojisLoaded$1.java	2021-12-24 01:10:12.783026500 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreEmojiGuild$handleGuildEmojisLoaded$1.java	2021-12-24 01:08:33.279263300 +0400
@@ -23,7 +23,7 @@ public final class StoreEmojiGuild$handl
 
     @Override // kotlin.jvm.functions.Function0
     public final void invoke() {
-        this.this$0.guildEmoji.put(Long.valueOf(this.$guildId), this.$emojis);
+        StoreEmojiGuild.access$getGuildEmoji$p(this.this$0).put(Long.valueOf(this.$guildId), this.$emojis);
         this.this$0.markChanged();
     }
 }
