--- ./discord_94011/src/main/java/com/discord/stores/StoreMFA$togglingSMSBackup$1.java	2021-12-24 01:10:12.876026100 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreMFA$togglingSMSBackup$1.java	2021-12-24 01:08:33.400262600 +0400
@@ -20,7 +20,7 @@ public final class StoreMFA$togglingSMSB
     @Override // kotlin.jvm.functions.Function0
     public final void invoke() {
         StoreMFA storeMFA = this.this$0;
-        storeMFA.state = StoreMFA.State.copy$default(storeMFA.state, null, true, 1, null);
+        StoreMFA.access$setState$p(storeMFA, StoreMFA.State.copy$default(StoreMFA.access$getState$p(storeMFA), null, true, 1, null));
         this.this$0.markChanged();
     }
 }
