--- ./discord_94011/src/main/java/com/discord/stores/StoreGuildProfiles$fetchGuildProfile$1.java	2021-12-24 01:10:12.830023300 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreGuildProfiles$fetchGuildProfile$1.java	2021-12-24 01:08:33.329263200 +0400
@@ -28,9 +28,9 @@ public final class StoreGuildProfiles$fe
 
     @Override // kotlin.jvm.functions.Function0
     public final void invoke() {
-        StoreGuildProfiles.GuildProfileData guildProfileData = (StoreGuildProfiles.GuildProfileData) this.this$0.guildProfilesState.get(Long.valueOf(this.$guildId));
+        StoreGuildProfiles.GuildProfileData guildProfileData = (StoreGuildProfiles.GuildProfileData) StoreGuildProfiles.access$getGuildProfilesState$p(this.this$0).get(Long.valueOf(this.$guildId));
         if ((guildProfileData != null ? guildProfileData.getFetchState() : null) != StoreGuildProfiles.FetchStates.FETCHING) {
-            this.this$0.handleGuildProfileFetchStart(this.$guildId);
+            StoreGuildProfiles.access$handleGuildProfileFetchStart(this.this$0, this.$guildId);
             ObservableExtensionsKt.appSubscribe$default(ObservableExtensionsKt.restSubscribeOn$default(RestAPI.Companion.getApi().getGuildPreview(this.$guildId), false, 1, null), this.this$0.getClass(), (Context) null, (Function1) null, new Function1<Error, Unit>(this) {
                 /* class com.discord.stores.StoreGuildProfiles$fetchGuildProfile$1.AnonymousClass2 */
                 public final /* synthetic */ StoreGuildProfiles$fetchGuildProfile$1 this$0;
@@ -49,7 +49,7 @@ public final class StoreGuildProfiles$fe
 
                 public final void invoke(Error error) {
                     m.checkNotNullParameter(error, "it");
-                    this.this$0.this$0.dispatcher.schedule(new Function0<Unit>(this) {
+                    StoreGuildProfiles.access$getDispatcher$p(this.this$0.this$0).schedule(new Function0<Unit>(this) {
                         /* class com.discord.stores.StoreGuildProfiles$fetchGuildProfile$1.AnonymousClass2.AnonymousClass1 */
                         public final /* synthetic */ AnonymousClass2 this$0;
 
@@ -60,7 +60,7 @@ public final class StoreGuildProfiles$fe
                         @Override // kotlin.jvm.functions.Function0
                         public final void invoke() {
                             StoreGuildProfiles$fetchGuildProfile$1 storeGuildProfiles$fetchGuildProfile$1 = this.this$0.this$0;
-                            storeGuildProfiles$fetchGuildProfile$1.this$0.handleGuildProfileFetchFailed(storeGuildProfiles$fetchGuildProfile$1.$guildId);
+                            StoreGuildProfiles.access$handleGuildProfileFetchFailed(storeGuildProfiles$fetchGuildProfile$1.this$0, storeGuildProfiles$fetchGuildProfile$1.$guildId);
                         }
                     });
                 }
@@ -82,7 +82,7 @@ public final class StoreGuildProfiles$fe
 
                 public final void invoke(final GuildPreview guildPreview) {
                     m.checkNotNullParameter(guildPreview, "guildPreview");
-                    this.this$0.this$0.dispatcher.schedule(new Function0<Unit>(this) {
+                    StoreGuildProfiles.access$getDispatcher$p(this.this$0.this$0).schedule(new Function0<Unit>(this) {
                         /* class com.discord.stores.StoreGuildProfiles$fetchGuildProfile$1.AnonymousClass1.AnonymousClass1 */
                         public final /* synthetic */ AnonymousClass1 this$0;
 
@@ -92,7 +92,7 @@ public final class StoreGuildProfiles$fe
 
                         @Override // kotlin.jvm.functions.Function0
                         public final void invoke() {
-                            this.this$0.this$0.this$0.handleGuildProfileFetchSuccess(guildPreview);
+                            StoreGuildProfiles.access$handleGuildProfileFetchSuccess(this.this$0.this$0.this$0, guildPreview);
                         }
                     });
                 }
