--- ./discord_94011/src/main/java/com/discord/stores/StoreEmojiGuild$deleteEmoji$1.java	2021-12-24 01:10:12.783026500 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreEmojiGuild$deleteEmoji$1.java	2021-12-24 01:08:33.278262200 +0400
@@ -27,9 +27,9 @@ public final class StoreEmojiGuild$delet
 
     @Override // kotlin.jvm.functions.Function0
     public final void invoke() {
-        Map map = this.this$0.guildEmoji;
+        Map access$getGuildEmoji$p = StoreEmojiGuild.access$getGuildEmoji$p(this.this$0);
         Long valueOf = Long.valueOf(this.$guildId);
-        List list = (List) this.this$0.guildEmoji.get(Long.valueOf(this.$guildId));
+        List list = (List) StoreEmojiGuild.access$getGuildEmoji$p(this.this$0).get(Long.valueOf(this.$guildId));
         if (list == null) {
             list = n.emptyList();
         }
@@ -39,7 +39,7 @@ public final class StoreEmojiGuild$delet
                 arrayList.add(obj);
             }
         }
-        map.put(valueOf, arrayList);
+        access$getGuildEmoji$p.put(valueOf, arrayList);
         this.this$0.markChanged();
     }
 }
