--- ./discord_94011/src/main/java/com/discord/stores/StoreEntitlements$fetchMyEntitlementsForApplication$3.java	2021-12-24 01:10:12.785024100 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreEntitlements$fetchMyEntitlementsForApplication$3.java	2021-12-24 01:08:33.280262500 +0400
@@ -32,7 +32,7 @@ public final class StoreEntitlements$fet
 
     public final void invoke(final List<ModelEntitlement> list) {
         m.checkNotNullParameter(list, "entitlements");
-        this.this$0.dispatcher.schedule(new Function0<Unit>(this) {
+        StoreEntitlements.access$getDispatcher$p(this.this$0).schedule(new Function0<Unit>(this) {
             /* class com.discord.stores.StoreEntitlements$fetchMyEntitlementsForApplication$3.AnonymousClass1 */
             public final /* synthetic */ StoreEntitlements$fetchMyEntitlementsForApplication$3 this$0;
 
