--- ./discord_94011/src/main/java/com/discord/stores/StoreExpressionPickerNavigation$onSelectTab$1.java	2021-12-24 01:10:12.794027300 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreExpressionPickerNavigation$onSelectTab$1.java	2021-12-24 01:08:33.291262000 +0400
@@ -21,7 +21,7 @@ public final class StoreExpressionPicker
 
     @Override // kotlin.jvm.functions.Function0
     public final void invoke() {
-        StoreExpressionPickerNavigation.access$setSelectedTab$p(this.this$0, this.$expressionTrayTab);
+        this.this$0.selectedTab = this.$expressionTrayTab;
         this.this$0.markChanged();
     }
 }
