--- ./discord_94011/src/main/java/com/discord/stores/StoreCalls$call$1.java	2021-12-24 01:10:12.742025200 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreCalls$call$1.java	2021-12-24 01:08:33.238262700 +0400
@@ -27,7 +27,7 @@ public final class StoreCalls$call$1 ext
     }
 
     public final void invoke(boolean z2) {
-        this.this$0.stream.getVoiceChannelSelected$app_productionGoogleRelease().selectVoiceChannel(this.$channelId);
+        StoreCalls.access$getStream$p(this.this$0).getVoiceChannelSelected$app_productionGoogleRelease().selectVoiceChannel(this.$channelId);
         if (z2) {
             StoreCalls.ring$default(this.this$0, this.$channelId, null, 2, null);
         }
