--- ./discord_94011/src/main/java/com/discord/stores/StoreApplicationCommands$requestApplications$1.java	2021-12-24 01:10:12.715027600 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreApplicationCommands$requestApplications$1.java	2021-12-24 01:08:33.210262000 +0400
@@ -22,17 +22,17 @@ public final class StoreApplicationComma
 
     @Override // kotlin.jvm.functions.Function0
     public final void invoke() {
-        String access$generateNonce = StoreApplicationCommands.access$generateNonce(this.this$0);
-        StoreApplicationCommands.access$setApplicationNonce$p(this.this$0, access$generateNonce);
-        StoreApplicationCommands.access$setApplications$p(this.this$0, n.emptyList());
-        StoreApplicationCommands.access$getApplicationCommandIndexes$p(this.this$0).clear();
+        String generateNonce = this.this$0.generateNonce();
+        this.this$0.applicationNonce = generateNonce;
+        this.this$0.applications = n.emptyList();
+        this.this$0.applicationCommandIndexes.clear();
         this.this$0.markChanged(StoreApplicationCommands.Companion.getGuildApplicationsUpdate());
         Long l = this.$guildId;
         if (l == null || l.longValue() <= 0) {
             StoreApplicationCommands storeApplicationCommands = this.this$0;
-            StoreApplicationCommands.access$handleGuildApplicationsUpdate(storeApplicationCommands, m.listOf(StoreApplicationCommands.access$getBuiltInCommandsProvider$p(storeApplicationCommands).getBuiltInApplication()));
+            storeApplicationCommands.handleGuildApplicationsUpdate(m.listOf(storeApplicationCommands.builtInCommandsProvider.getBuiltInApplication()));
             return;
         }
-        StoreGatewayConnection.requestApplicationCommands$default(StoreApplicationCommands.access$getStoreGatewayConnection$p(this.this$0), this.$guildId.longValue(), access$generateNonce, true, null, null, 0, 24, null);
+        StoreGatewayConnection.requestApplicationCommands$default(this.this$0.storeGatewayConnection, this.$guildId.longValue(), generateNonce, true, null, null, 0, 24, null);
     }
 }
