--- ./discord_94011/src/main/java/com/discord/stores/StoreApplicationCommands$requestInitialApplicationCommands$$inlined$let$lambda$1.java	2021-12-24 01:10:12.709023300 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreApplicationCommands$requestInitialApplicationCommands$$inlined$let$lambda$1.java	2021-12-24 01:08:33.204263500 +0400
@@ -22,6 +22,6 @@ public final class StoreApplicationComma
     @Override // kotlin.jvm.functions.Function0
     public final void invoke() {
         StoreApplicationCommands storeApplicationCommands = this.this$0;
-        StoreApplicationCommands.access$handleDiscoverCommandsUpdate(storeApplicationCommands, StoreApplicationCommands.access$getBuiltInCommandsProvider$p(storeApplicationCommands).getBuiltInCommands());
+        storeApplicationCommands.handleDiscoverCommandsUpdate(storeApplicationCommands.builtInCommandsProvider.getBuiltInCommands());
     }
 }
