--- ./discord_94011/src/main/java/com/discord/stores/StoreExpressionPickerNavigation$observeSelectedTab$1.java	2021-12-24 01:10:12.794027300 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreExpressionPickerNavigation$observeSelectedTab$1.java	2021-12-24 01:08:33.291262000 +0400
@@ -18,6 +18,6 @@ public final class StoreExpressionPicker
 
     @Override // kotlin.jvm.functions.Function0
     public final ExpressionTrayTab invoke() {
-        return StoreExpressionPickerNavigation.access$getSelectedTab$p(this.this$0);
+        return this.this$0.selectedTab;
     }
 }
