--- ./discord_94011/src/main/java/com/discord/stores/StoreEntitlements$fetchMyGiftEntitlements$2.java	2021-12-24 01:10:12.785024100 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreEntitlements$fetchMyGiftEntitlements$2.java	2021-12-24 01:08:33.281264300 +0400
@@ -29,7 +29,7 @@ public final class StoreEntitlements$fet
 
     public final void invoke(Error error) {
         m.checkNotNullParameter(error, "it");
-        this.this$0.dispatcher.schedule(new Function0<Unit>(this) {
+        StoreEntitlements.access$getDispatcher$p(this.this$0).schedule(new Function0<Unit>(this) {
             /* class com.discord.stores.StoreEntitlements$fetchMyGiftEntitlements$2.AnonymousClass1 */
             public final /* synthetic */ StoreEntitlements$fetchMyGiftEntitlements$2 this$0;
 
