--- ./discord_94011/src/main/java/com/discord/stores/StoreGuildScheduledEvents.java	2021-12-24 01:10:12.836028100 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreGuildScheduledEvents.java	2021-12-24 01:08:33.334261200 +0400
@@ -67,6 +67,18 @@ public final class StoreGuildScheduledEv
         this.clock = clock2;
     }
 
+    public static final /* synthetic */ Dispatcher access$getDispatcher$p(StoreGuildScheduledEvents storeGuildScheduledEvents) {
+        return storeGuildScheduledEvents.dispatcher;
+    }
+
+    public static final /* synthetic */ HashMap access$getUserGuildScheduledEventIds$p(StoreGuildScheduledEvents storeGuildScheduledEvents) {
+        return storeGuildScheduledEvents.userGuildScheduledEventIds;
+    }
+
+    public static final /* synthetic */ HashSet access$getUserGuildScheduledEventsFetches$p(StoreGuildScheduledEvents storeGuildScheduledEvents) {
+        return storeGuildScheduledEvents.userGuildScheduledEventsFetches;
+    }
+
     private final void addUserRsvpForEvent(long j, long j2) {
         if (this.userGuildScheduledEventIds.containsKey(Long.valueOf(j))) {
             Set<Long> set = this.userGuildScheduledEventIds.get(Long.valueOf(j));
