--- ./discord_94011/src/main/java/com/discord/stores/StoreStickers$fetchStickerPack$1.java	2021-12-24 01:10:12.993025800 +0400
+++ ./bluecord/src/main/java/com/discord/stores/StoreStickers$fetchStickerPack$1.java	2021-12-24 01:08:33.511261600 +0400
@@ -26,7 +26,7 @@ public final class StoreStickers$fetchSt
     @Override // kotlin.jvm.functions.Function0
     public final void invoke() {
         this.this$0.handleNewLoadingStickerPacks(m.listOf(Long.valueOf(this.$stickerPackId)));
-        ObservableExtensionsKt.appSubscribe$default(ObservableExtensionsKt.restSubscribeOn$default(this.this$0.api.getStickerPack(this.$stickerPackId), false, 1, null), this.this$0.getClass(), (Context) null, (Function1) null, (Function1) null, (Function0) null, (Function0) null, new Function1<ModelStickerPack, Unit>(this) {
+        ObservableExtensionsKt.appSubscribe$default(ObservableExtensionsKt.restSubscribeOn$default(StoreStickers.access$getApi$p(this.this$0).getStickerPack(this.$stickerPackId), false, 1, null), this.this$0.getClass(), (Context) null, (Function1) null, (Function1) null, (Function0) null, (Function0) null, new Function1<ModelStickerPack, Unit>(this) {
             /* class com.discord.stores.StoreStickers$fetchStickerPack$1.AnonymousClass1 */
             public final /* synthetic */ StoreStickers$fetchStickerPack$1 this$0;
 
@@ -44,7 +44,7 @@ public final class StoreStickers$fetchSt
 
             public final void invoke(final ModelStickerPack modelStickerPack) {
                 d0.z.d.m.checkNotNullParameter(modelStickerPack, "newPack");
-                this.this$0.this$0.dispatcher.schedule(new Function0<Unit>(this) {
+                StoreStickers.access$getDispatcher$p(this.this$0.this$0).schedule(new Function0<Unit>(this) {
                     /* class com.discord.stores.StoreStickers$fetchStickerPack$1.AnonymousClass1.AnonymousClass1 */
                     public final /* synthetic */ AnonymousClass1 this$0;
 
