--- ./discord_94011/src/main/java/com/discord/widgets/chat/input/WidgetChatInputEditText.java	2021-12-24 01:10:13.946028400 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/chat/input/WidgetChatInputEditText.java	2021-12-24 01:08:34.355264300 +0400
@@ -8,6 +8,7 @@ import kotlin.Metadata;
 import kotlin.Unit;
 import kotlin.jvm.functions.Function0;
 import kotlin.jvm.internal.DefaultConstructorMarker;
+import mods.ThemingTools;
 import rx.subjects.BehaviorSubject;
 import rx.subjects.Subject;
 
@@ -53,6 +54,7 @@ public final class WidgetChatInputEditTe
         setOnTextChangedListener();
         setSoftwareKeyboardSendBehavior();
         setHardwareKeyboardSendBehavior();
+        ThemingTools.setTrayText(this.editText);
     }
 
     private final void setHardwareKeyboardSendBehavior() {
