--- ./discord_94011/src/main/java/androidx/lifecycle/LifecycleController$observer$1.java	2021-12-24 01:10:09.130025800 +0400
+++ ./bluecord/src/main/java/androidx/lifecycle/LifecycleController$observer$1.java	2021-12-24 01:08:29.968265600 +0400
@@ -31,10 +31,10 @@ public final class LifecycleController$o
         }
         Lifecycle lifecycle2 = lifecycleOwner.getLifecycle();
         m.checkNotNullExpressionValue(lifecycle2, "source.lifecycle");
-        if (lifecycle2.getCurrentState().compareTo((Enum) LifecycleController.access$getMinState$p(this.this$0)) < 0) {
-            LifecycleController.access$getDispatchQueue$p(this.this$0).pause();
+        if (lifecycle2.getCurrentState().compareTo((Enum) this.this$0.minState) < 0) {
+            this.this$0.dispatchQueue.pause();
         } else {
-            LifecycleController.access$getDispatchQueue$p(this.this$0).resume();
+            this.this$0.dispatchQueue.resume();
         }
     }
 }
