--- ./discord_94011/src/main/java/com/discord/widgets/directories/WidgetDirectoryCategory$onViewBoundOrOnResume$1.java	2021-12-24 01:10:14.260024400 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/directories/WidgetDirectoryCategory$onViewBoundOrOnResume$1.java	2021-12-24 01:08:34.849263700 +0400
@@ -28,6 +28,6 @@ public final class WidgetDirectoryCatego
 
     public final void invoke(WidgetDirectoriesViewModel.ViewState viewState) {
         m.checkNotNullParameter(viewState, "it");
-        this.this$0.configureUi(viewState);
+        WidgetDirectoryCategory.access$configureUi(this.this$0, viewState);
     }
 }
