--- ./discord_94011/src/main/java/com/discord/widgets/directories/WidgetDirectoryCategory.java	2021-12-24 01:10:14.257024000 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/directories/WidgetDirectoryCategory.java	2021-12-24 01:08:34.846264800 +0400
@@ -57,7 +57,11 @@ public final class WidgetDirectoryCatego
         super(2131559039);
     }
 
-    public final void configureUi(WidgetDirectoriesViewModel.ViewState viewState) {
+    public static final /* synthetic */ void access$configureUi(WidgetDirectoryCategory widgetDirectoryCategory, WidgetDirectoriesViewModel.ViewState viewState) {
+        widgetDirectoryCategory.configureUi(viewState);
+    }
+
+    private final void configureUi(WidgetDirectoriesViewModel.ViewState viewState) {
         List<DirectoryEntryData> list;
         if (getArgs().getDirectoryCategory().ordinal() != 0) {
             list = viewState.getDirectoryEntryData().get(getArgs().getDirectoryCategory());
