--- ./discord_94011/src/main/java/com/discord/app/AppActivity.java	2021-12-24 01:10:12.284027400 +0400
+++ ./bluecord/src/main/java/com/discord/app/AppActivity.java	2021-12-24 01:08:32.825263500 +0400
@@ -71,6 +71,9 @@ import kotlin.Metadata;
 import kotlin.Unit;
 import kotlin.jvm.functions.Function0;
 import kotlin.jvm.functions.Function1;
+import mods.ThemingTools;
+import mods.activity.BlueSettingsActivity;
+import mods.anti.AntiDiscordRebrand;
 import rx.subjects.PublishSubject;
 import rx.subjects.Subject;
 
@@ -606,6 +609,7 @@ public class AppActivity extends c.a.d.d
     @Override // androidx.activity.ComponentActivity, androidx.core.app.ComponentActivity, androidx.appcompat.app.AppCompatActivity, androidx.fragment.app.FragmentActivity
     public void onCreate(Bundle bundle) {
         super.onCreate(bundle);
+        ThemingTools.init(this, false);
         a aVar = new a(0, this);
         a aVar2 = new a(1, this);
         try {
@@ -619,7 +623,7 @@ public class AppActivity extends c.a.d.d
                     i = 2131951650;
                 }
             }
-            setTheme(i);
+            setTheme(AntiDiscordRebrand.overrideTheme(i));
             b(companion.getUserSettingsSystem().getLocale(), false);
             aVar.m1invoke();
             aVar2.m1invoke();
@@ -666,6 +670,7 @@ public class AppActivity extends c.a.d.d
     @Override // com.discord.app.AppTransitionActivity, androidx.fragment.app.FragmentActivity
     public void onResume() {
         super.onResume();
+        BlueSettingsActivity.refreshIfNeeded(this);
         j jVar = j.g;
         m.checkNotNullParameter(this, "activity");
         boolean booleanExtra = c().getBooleanExtra("INTENT_RECREATE", false);
