--- ./discord_94011/src/main/java/com/discord/widgets/channels/threads/WidgetThreadDraftArchiveSheet$bindSubscriptions$1.java	2021-12-24 01:10:13.882025200 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/channels/threads/WidgetThreadDraftArchiveSheet$bindSubscriptions$1.java	2021-12-24 01:08:34.263263000 +0400
@@ -24,6 +24,6 @@ public final /* synthetic */ class Widge
 
     public final void invoke(WidgetThreadDraftArchiveSheet.Model model) {
         m.checkNotNullParameter(model, "p1");
-        WidgetThreadDraftArchiveSheet.access$configureUI((WidgetThreadDraftArchiveSheet) this.receiver, model);
+        ((WidgetThreadDraftArchiveSheet) this.receiver).configureUI(model);
     }
 }
