--- ./discord_94011/src/main/java/com/discord/widgets/stage/sheet/WidgetStageModeratorJoinBottomSheet.java	2021-12-24 01:10:15.326027300 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/stage/sheet/WidgetStageModeratorJoinBottomSheet.java	2021-12-24 01:08:35.800261800 +0400
@@ -55,6 +55,10 @@ public final class WidgetStageModeratorJ
         super(false, 1, null);
     }
 
+    public static final /* synthetic */ Bundle access$getArgumentsOrDefault$p(WidgetStageModeratorJoinBottomSheet widgetStageModeratorJoinBottomSheet) {
+        return widgetStageModeratorJoinBottomSheet.getArgumentsOrDefault();
+    }
+
     private final WidgetStageModeratorJoinBottomSheetBinding getBinding() {
         return (WidgetStageModeratorJoinBottomSheetBinding) this.binding$delegate.getValue((Fragment) this, $$delegatedProperties[0]);
     }
