--- ./discord_94011/src/main/java/com/discord/widgets/settings/connections/WidgetSettingsUserConnectionsAddXbox.java	2021-12-24 01:10:15.179026700 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/settings/connections/WidgetSettingsUserConnectionsAddXbox.java	2021-12-24 01:08:35.675261500 +0400
@@ -52,8 +52,19 @@ public final class WidgetSettingsUserCon
         super(2131559030);
     }
 
-    /* access modifiers changed from: private */
-    public final WidgetConnectionsAddXboxBinding getBinding() {
+    public static final /* synthetic */ WidgetConnectionsAddXboxBinding access$getBinding$p(WidgetSettingsUserConnectionsAddXbox widgetSettingsUserConnectionsAddXbox) {
+        return widgetSettingsUserConnectionsAddXbox.getBinding();
+    }
+
+    public static final /* synthetic */ void access$showPinError(WidgetSettingsUserConnectionsAddXbox widgetSettingsUserConnectionsAddXbox) {
+        widgetSettingsUserConnectionsAddXbox.showPinError();
+    }
+
+    public static final /* synthetic */ void access$trackXboxLinkFailed(WidgetSettingsUserConnectionsAddXbox widgetSettingsUserConnectionsAddXbox, Error error) {
+        widgetSettingsUserConnectionsAddXbox.trackXboxLinkFailed(error);
+    }
+
+    private final WidgetConnectionsAddXboxBinding getBinding() {
         return (WidgetConnectionsAddXboxBinding) this.binding$delegate.getValue((Fragment) this, $$delegatedProperties[0]);
     }
 
@@ -61,14 +72,12 @@ public final class WidgetSettingsUserCon
         Companion.launch(context);
     }
 
-    /* access modifiers changed from: private */
-    public final void showPinError() {
+    private final void showPinError() {
         c.a.d.m.g(getContext(), 2131887619, 0, null, 12);
         getBinding().b.b();
     }
 
-    /* access modifiers changed from: private */
-    public final void trackXboxLinkFailed(Error error) {
+    private final void trackXboxLinkFailed(Error error) {
         AnalyticsTracker analyticsTracker = AnalyticsTracker.INSTANCE;
         String bodyText = error.getBodyText();
         Error.Response response = error.getResponse();
