--- ./discord_94011/src/main/java/com/discord/widgets/chat/input/emoji/WidgetEmojiPicker$onViewBoundOrOnResume$2.java	2021-12-24 01:10:13.994027100 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/chat/input/emoji/WidgetEmojiPicker$onViewBoundOrOnResume$2.java	2021-12-24 01:08:34.398261100 +0400
@@ -28,6 +28,6 @@ public final class WidgetEmojiPicker$onV
 
     public final void invoke(EmojiPickerViewModel.Event event) {
         m.checkNotNullParameter(event, "event");
-        this.this$0.handleEvent(event);
+        WidgetEmojiPicker.access$handleEvent(this.this$0, event);
     }
 }
