--- ./discord_94011/src/main/java/com/discord/widgets/chat/input/emoji/WidgetEmojiPicker$setUpCategoryRecycler$1.java	2021-12-24 01:10:13.994027100 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/chat/input/emoji/WidgetEmojiPicker$setUpCategoryRecycler$1.java	2021-12-24 01:08:34.398261100 +0400
@@ -23,6 +23,6 @@ public final /* synthetic */ class Widge
 
     public final void invoke(EmojiCategoryItem emojiCategoryItem) {
         m.checkNotNullParameter(emojiCategoryItem, "p1");
-        ((WidgetEmojiPicker) this.receiver).onCategoryClicked(emojiCategoryItem);
+        WidgetEmojiPicker.access$onCategoryClicked((WidgetEmojiPicker) this.receiver, emojiCategoryItem);
     }
 }
