--- ./discord_94011/src/main/java/com/discord/models/message/Message.java	2021-12-24 01:10:12.658026500 +0400
+++ ./bluecord/src/main/java/com/discord/models/message/Message.java	2021-12-24 01:08:33.152262200 +0400
@@ -44,6 +44,7 @@ public final class Message {
     private final long channelId;
     private final List<Component> components;
     private final String content;
+    public boolean deleted;
     private final UtcDateTime editedTimestamp;
     private final List<MessageEmbed> embeds;
     private final Long flags;
