--- ./discord_94011/src/main/java/org/webrtc/RTCStats.java	2021-12-24 01:10:16.680026900 +0400
+++ ./bluecord/src/main/java/org/webrtc/RTCStats.java	2021-12-24 01:08:37.060262700 +0400
@@ -6,7 +6,7 @@ import java.util.Map;
 public class RTCStats {
 
     /* renamed from: id  reason: collision with root package name */
-    private final String f2960id;
+    private final String f2962id;
     private final Map<String, Object> members;
     private final long timestampUs;
     private final String type;
@@ -14,7 +14,7 @@ public class RTCStats {
     public RTCStats(long j, String str, String str2, Map<String, Object> map) {
         this.timestampUs = j;
         this.type = str;
-        this.f2960id = str2;
+        this.f2962id = str2;
         this.members = map;
     }
 
@@ -44,7 +44,7 @@ public class RTCStats {
     }
 
     public String getId() {
-        return this.f2960id;
+        return this.f2962id;
     }
 
     public Map<String, Object> getMembers() {
@@ -65,7 +65,7 @@ public class RTCStats {
         P.append(", type: ");
         P.append(this.type);
         P.append(", id: ");
-        P.append(this.f2960id);
+        P.append(this.f2962id);
         for (Map.Entry<String, Object> entry : this.members.entrySet()) {
             P.append(", ");
             P.append(entry.getKey());
