--- ./discord_94011/src/main/java/org/webrtc/StatsReport.java	2021-12-24 01:10:16.686027000 +0400
+++ ./bluecord/src/main/java/org/webrtc/StatsReport.java	2021-12-24 01:08:37.066262100 +0400
@@ -5,7 +5,7 @@ import c.d.b.a.a;
 public class StatsReport {
 
     /* renamed from: id  reason: collision with root package name */
-    public final String f2962id;
+    public final String f2964id;
     public final double timestamp;
     public final String type;
     public final Value[] values;
@@ -30,7 +30,7 @@ public class StatsReport {
 
     @CalledByNative
     public StatsReport(String str, String str2, double d, Value[] valueArr) {
-        this.f2962id = str;
+        this.f2964id = str;
         this.type = str2;
         this.timestamp = d;
         this.values = valueArr;
@@ -38,7 +38,7 @@ public class StatsReport {
 
     public String toString() {
         StringBuilder P = a.P("id: ");
-        P.append(this.f2962id);
+        P.append(this.f2964id);
         P.append(", type: ");
         P.append(this.type);
         P.append(", timestamp: ");
