--- ./discord_94011/src/main/java/org/webrtc/TextureBufferImpl.java	2021-12-24 01:10:16.687025200 +0400
+++ ./bluecord/src/main/java/org/webrtc/TextureBufferImpl.java	2021-12-24 01:08:37.067263400 +0400
@@ -11,7 +11,7 @@ public class TextureBufferImpl implement
     private final int height;
 
     /* renamed from: id  reason: collision with root package name */
-    private final int f2963id;
+    private final int f2965id;
     private final RefCountDelegate refCountDelegate;
     private final RefCountMonitor refCountMonitor;
     private final Handler toI420Handler;
@@ -76,7 +76,7 @@ public class TextureBufferImpl implement
 
     @Override // org.webrtc.VideoFrame.TextureBuffer
     public int getTextureId() {
-        return this.f2963id;
+        return this.f2965id;
     }
 
     public Handler getToI420Handler() {
@@ -135,7 +135,7 @@ public class TextureBufferImpl implement
         Matrix matrix2 = new Matrix(this.transformMatrix);
         matrix2.preConcat(matrix);
         retain();
-        return new TextureBufferImpl(i, i2, i3, i4, this.type, this.f2963id, matrix2, this.toI420Handler, this.yuvConverter, new RefCountMonitor() {
+        return new TextureBufferImpl(i, i2, i3, i4, this.type, this.f2965id, matrix2, this.toI420Handler, this.yuvConverter, new RefCountMonitor() {
             /* class org.webrtc.TextureBufferImpl.AnonymousClass2 */
 
             @Override // org.webrtc.TextureBufferImpl.RefCountMonitor
@@ -161,7 +161,7 @@ public class TextureBufferImpl implement
         this.width = i3;
         this.height = i4;
         this.type = type2;
-        this.f2963id = i5;
+        this.f2965id = i5;
         this.transformMatrix = matrix;
         this.toI420Handler = handler;
         this.yuvConverter = yuvConverter2;
