--- ./discord_94011/src/main/java/com/discord/widgets/guilds/invite/InviteGenerator$generateForAppComponent$1.java	2021-12-24 01:10:14.432026100 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/guilds/invite/InviteGenerator$generateForAppComponent$1.java	2021-12-24 01:08:35.036260700 +0400
@@ -28,6 +28,6 @@ public final class InviteGenerator$gener
 
     public final void invoke(ModelInvite modelInvite) {
         m.checkNotNullParameter(modelInvite, "invite");
-        this.this$0.handleGeneratedInvite(modelInvite);
+        InviteGenerator.access$handleGeneratedInvite(this.this$0, modelInvite);
     }
 }
