--- ./discord_94011/src/main/java/com/discord/utilities/rest/AttachmentRequestBody.java	2021-12-24 01:10:13.310023800 +0400
+++ ./bluecord/src/main/java/com/discord/utilities/rest/AttachmentRequestBody.java	2021-12-24 01:08:33.825261400 +0400
@@ -47,7 +47,7 @@ public final class AttachmentRequestBody
 
     @Override // okhttp3.RequestBody
     public MediaType contentType() {
-        MediaType.a aVar = MediaType.f2950c;
+        MediaType.a aVar = MediaType.f2952c;
         return MediaType.a.b(AttachmentUtilsKt.getMimeType(this.attachment, this.contentResolver));
     }
 
