--- ./discord_94011/src/main/java/com/discord/utilities/rest/RestAPI$Companion$buildAnalyticsInterceptor$$inlined$invoke$1.java	2021-12-24 01:10:13.311024800 +0400
+++ ./bluecord/src/main/java/com/discord/utilities/rest/RestAPI$Companion$buildAnalyticsInterceptor$$inlined$invoke$1.java	2021-12-24 01:08:33.826270400 +0400
@@ -27,7 +27,7 @@ public final class RestAPI$Companion$bui
         m.checkParameterIsNotNull(c2, "request");
         new LinkedHashMap();
         w wVar = c2.b;
-        String str = c2.f2953c;
+        String str = c2.f2955c;
         RequestBody requestBody = c2.e;
         if (c2.f.isEmpty()) {
             map = new LinkedHashMap();
