--- ./discord_94011/src/main/java/i0/e0/b/a.java	2021-12-24 01:10:16.501028200 +0400
+++ ./bluecord/src/main/java/i0/e0/b/a.java	2021-12-24 01:08:36.874263500 +0400
@@ -11,7 +11,7 @@ public final class a<T> implements h<T,
     public static final MediaType b = MediaType.a.a("text/plain; charset=UTF-8");
 
     static {
-        MediaType.a aVar = MediaType.f2950c;
+        MediaType.a aVar = MediaType.f2952c;
     }
 
     /* Return type fixed from 'java.lang.Object' to match base method */
