--- ./discord_94011/src/main/java/com/discord/widgets/guild_role_subscriptions/setup/WidgetGuildRoleSubscriptionPlanDetails$onViewBound$1.java	2021-12-24 01:10:14.334025900 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/guild_role_subscriptions/setup/WidgetGuildRoleSubscriptionPlanDetails$onViewBound$1.java	2021-12-24 01:08:34.926262500 +0400
@@ -27,6 +27,6 @@ public final class WidgetGuildRoleSubscr
 
     public final void invoke(String str) {
         m.checkNotNullParameter(str, "it");
-        WidgetGuildRoleSubscriptionPlanDetails.access$getViewModel$p(this.this$0).setDescription(str);
+        this.this$0.getViewModel().setDescription(str);
     }
 }
