--- ./discord_94011/src/main/java/com/discord/widgets/guild_role_subscriptions/setup/WidgetGuildRoleSubscriptionPlanDetails$onViewBoundOrOnResume$1.java	2021-12-24 01:10:14.334025900 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/guild_role_subscriptions/setup/WidgetGuildRoleSubscriptionPlanDetails$onViewBoundOrOnResume$1.java	2021-12-24 01:08:34.927262500 +0400
@@ -28,6 +28,6 @@ public final class WidgetGuildRoleSubscr
 
     public final void invoke(GuildRoleSubscriptionPlanDetailsViewModel.ViewState viewState) {
         m.checkNotNullParameter(viewState, "it");
-        WidgetGuildRoleSubscriptionPlanDetails.access$configureUI(this.this$0, viewState);
+        this.this$0.configureUI(viewState);
     }
 }
