--- ./discord_94011/src/main/java/com/discord/widgets/guild_role_subscriptions/setup/WidgetGuildRoleSubscriptionPlanSetup$onViewBound$2.java	2021-12-24 01:10:14.339026300 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/guild_role_subscriptions/setup/WidgetGuildRoleSubscriptionPlanSetup$onViewBound$2.java	2021-12-24 01:08:34.939263100 +0400
@@ -12,9 +12,33 @@ public final class WidgetGuildRoleSubscr
         this.this$0 = widgetGuildRoleSubscriptionPlanSetup;
     }
 
+    /*  JADX ERROR: JadxRuntimeException in pass: InlineMethods
+        jadx.core.utils.exceptions.JadxRuntimeException: Failed to process method for inline: com.discord.widgets.guild_role_subscriptions.setup.WidgetGuildRoleSubscriptionPlanSetup.access$getSetupViewModel$p(com.discord.widgets.guild_role_subscriptions.setup.WidgetGuildRoleSubscriptionPlanSetup):com.discord.widgets.guild_role_subscriptions.setup.GuildRoleSubscriptionPlanSetupViewModel
+        	at jadx.core.dex.visitors.InlineMethods.processInvokeInsn(InlineMethods.java:68)
+        	at jadx.core.dex.visitors.InlineMethods.visit(InlineMethods.java:43)
+        Caused by: java.util.ConcurrentModificationException
+        	at java.base/java.util.ArrayList.removeIf(ArrayList.java:1685)
+        	at java.base/java.util.ArrayList.removeIf(ArrayList.java:1660)
+        	at jadx.core.dex.instructions.args.SSAVar.removeUse(SSAVar.java:125)
+        	at jadx.core.utils.InsnRemover.unbindArgUsage(InsnRemover.java:151)
+        	at jadx.core.dex.nodes.InsnNode.replaceArg(InsnNode.java:136)
+        	at jadx.core.dex.nodes.InsnNode.replaceArg(InsnNode.java:140)
+        	at jadx.core.dex.visitors.MarkMethodsForInline.addInlineAttr(MarkMethodsForInline.java:103)
+        	at jadx.core.dex.visitors.MarkMethodsForInline.inlineMth(MarkMethodsForInline.java:84)
+        	at jadx.core.dex.visitors.MarkMethodsForInline.process(MarkMethodsForInline.java:58)
+        	at jadx.core.dex.visitors.InlineMethods.processInvokeInsn(InlineMethods.java:57)
+        	... 1 more
+        */
     @Override // rx.functions.Func0, java.util.concurrent.Callable
-    public final Boolean call() {
-        this.this$0.getSetupViewModel().goToPreviousStep();
-        return Boolean.TRUE;
+    public final java.lang.Boolean call() {
+        /*
+            r1 = this;
+            com.discord.widgets.guild_role_subscriptions.setup.WidgetGuildRoleSubscriptionPlanSetup r0 = r1.this$0
+            com.discord.widgets.guild_role_subscriptions.setup.GuildRoleSubscriptionPlanSetupViewModel r0 = com.discord.widgets.guild_role_subscriptions.setup.WidgetGuildRoleSubscriptionPlanSetup.access$getSetupViewModel$p(r0)
+            r0.goToPreviousStep()
+            java.lang.Boolean r0 = java.lang.Boolean.TRUE
+            return r0
+        */
+        throw new UnsupportedOperationException("Method not decompiled: com.discord.widgets.guild_role_subscriptions.setup.WidgetGuildRoleSubscriptionPlanSetup$onViewBound$2.call():java.lang.Boolean");
     }
 }
