--- ./discord_94011/src/main/java/okhttp3/MultipartBody.java	2021-12-24 01:10:16.616025600 +0400
+++ ./bluecord/src/main/java/okhttp3/MultipartBody.java	2021-12-24 01:08:37.024260800 +0400
@@ -21,7 +21,7 @@ public final class MultipartBody extends
     public static final MediaType b = MediaType.a.a("multipart/form-data");
 
     /* renamed from: c  reason: collision with root package name */
-    public static final byte[] f2951c = {(byte) 58, (byte) 32};
+    public static final byte[] f2953c = {(byte) 58, (byte) 32};
     public static final byte[] d = {(byte) 13, (byte) 10};
     public static final byte[] e;
     public static final b f = new b(null);
@@ -100,7 +100,7 @@ public final class MultipartBody extends
         public MediaType b = MultipartBody.a;
 
         /* renamed from: c  reason: collision with root package name */
-        public final List<Part> f2952c = new ArrayList();
+        public final List<Part> f2954c = new ArrayList();
 
         public a() {
             String uuid = UUID.randomUUID().toString();
@@ -111,13 +111,13 @@ public final class MultipartBody extends
 
         public final a a(Part part) {
             m.checkParameterIsNotNull(part, "part");
-            this.f2952c.add(part);
+            this.f2954c.add(part);
             return this;
         }
 
         public final MultipartBody b() {
-            if (!this.f2952c.isEmpty()) {
-                return new MultipartBody(this.a, this.b, c.z(this.f2952c));
+            if (!this.f2954c.isEmpty()) {
+                return new MultipartBody(this.a, this.b, c.z(this.f2954c));
             }
             throw new IllegalStateException("Multipart body must have at least one part.".toString());
         }
@@ -159,7 +159,7 @@ public final class MultipartBody extends
     }
 
     static {
-        MediaType.a aVar = MediaType.f2950c;
+        MediaType.a aVar = MediaType.f2952c;
         MediaType.a.a("multipart/alternative");
         MediaType.a.a("multipart/digest");
         MediaType.a.a("multipart/parallel");
@@ -174,7 +174,7 @@ public final class MultipartBody extends
         this.i = byteString;
         this.j = mediaType;
         this.k = list;
-        MediaType.a aVar = MediaType.f2950c;
+        MediaType.a aVar = MediaType.f2952c;
         this.g = MediaType.a.a(mediaType + "; boundary=" + byteString.q());
     }
 
@@ -205,7 +205,7 @@ public final class MultipartBody extends
             if (headers != null) {
                 int size2 = headers.size();
                 for (int i3 = 0; i3 < size2; i3++) {
-                    bufferedSink.J(headers.d(i3)).write(f2951c).J(headers.g(i3)).write(d);
+                    bufferedSink.J(headers.d(i3)).write(f2953c).J(headers.g(i3)).write(d);
                 }
             }
             MediaType contentType = requestBody.contentType();
