--- ./discord_94011/src/main/java/okhttp3/Request.java	2021-12-24 01:10:16.616025600 +0400
+++ ./bluecord/src/main/java/okhttp3/Request.java	2021-12-24 01:08:37.024260800 +0400
@@ -23,7 +23,7 @@ public final class Request {
     public final w b;
 
     /* renamed from: c  reason: collision with root package name */
-    public final String f2953c;
+    public final String f2955c;
     public final Headers d;
     public final RequestBody e;
     public final Map<Class<?>, Object> f;
@@ -34,7 +34,7 @@ public final class Request {
         m.checkParameterIsNotNull(headers, "headers");
         m.checkParameterIsNotNull(map, "tags");
         this.b = wVar;
-        this.f2953c = str;
+        this.f2955c = str;
         this.d = headers;
         this.e = requestBody;
         this.f = map;
@@ -57,7 +57,7 @@ public final class Request {
 
     public String toString() {
         StringBuilder P = c.d.b.a.a.P("Request{method=");
-        P.append(this.f2953c);
+        P.append(this.f2955c);
         P.append(", url=");
         P.append(this.b);
         if (this.d.size() != 0) {
@@ -97,14 +97,14 @@ public final class Request {
         public String b;
 
         /* renamed from: c  reason: collision with root package name */
-        public Headers.a f2954c;
+        public Headers.a f2956c;
         public RequestBody d;
         public Map<Class<?>, Object> e;
 
         public a() {
             this.e = new LinkedHashMap();
             this.b = "GET";
-            this.f2954c = new Headers.a();
+            this.f2956c = new Headers.a();
         }
 
         public Request a() {
@@ -112,7 +112,7 @@ public final class Request {
             w wVar = this.a;
             if (wVar != null) {
                 String str = this.b;
-                Headers c2 = this.f2954c.c();
+                Headers c2 = this.f2956c.c();
                 RequestBody requestBody = this.d;
                 Map<Class<?>, Object> map2 = this.e;
                 byte[] bArr = c.a;
@@ -131,7 +131,7 @@ public final class Request {
         public a b(String str, String str2) {
             m.checkParameterIsNotNull(str, "name");
             m.checkParameterIsNotNull(str2, "value");
-            Headers.a aVar = this.f2954c;
+            Headers.a aVar = this.f2956c;
             Objects.requireNonNull(aVar);
             m.checkParameterIsNotNull(str, "name");
             m.checkParameterIsNotNull(str2, "value");
@@ -167,7 +167,7 @@ public final class Request {
 
         public a d(String str) {
             m.checkParameterIsNotNull(str, "name");
-            this.f2954c.d(str);
+            this.f2956c.d(str);
             return this;
         }
 
@@ -222,7 +222,7 @@ public final class Request {
             m.checkParameterIsNotNull(request, "request");
             this.e = new LinkedHashMap();
             this.a = request.b;
-            this.b = request.f2953c;
+            this.b = request.f2955c;
             this.d = request.e;
             if (request.f.isEmpty()) {
                 map = new LinkedHashMap<>();
@@ -230,7 +230,7 @@ public final class Request {
                 map = h0.toMutableMap(request.f);
             }
             this.e = map;
-            this.f2954c = request.d.e();
+            this.f2956c = request.d.e();
         }
     }
 }
