--- ./discord_94011/src/main/java/com/discord/widgets/mobile_reports/MobileReportsViewModel.java	2021-12-24 01:10:14.776026600 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/mobile_reports/MobileReportsViewModel.java	2021-12-24 01:08:35.270262600 +0400
@@ -1497,7 +1497,7 @@ public final class MobileReportsViewMode
                 Map<Long, String> channelNames = companion.getChannels().getChannelNames();
                 Map<Long, String> nickOrUsernames = MessageUtils.getNickOrUsernames(message, channel, map2, channel.n());
                 String content = message.getContent();
-                DraweeSpanStringBuilder parseChannelMessage = DiscordParser.parseChannelMessage(context2, content != null ? new Regex("\n").replace(content, " ") : null, new MessageRenderContext(context2, id2, false, nickOrUsernames, channelNames, map3, 0, null, null, 0, 0, null, null, null, 16320, null), new MessagePreprocessor(id2, null, null, false, 50, 6, null), DiscordParser.ParserOptions.REPLY, false);
+                DraweeSpanStringBuilder parseChannelMessage = DiscordParser.parseChannelMessage(context2, content != null ? new Regex("\n").replace(content, " ") : null, new MessageRenderContext(context2, id2, false, nickOrUsernames, channelNames, map3, 0, null, null, 0, 0, null, null, null, 16320, null), new MessagePreprocessor(id2, null, null, false, 50, 6, null), DiscordParser.ParserOptions.REPLY, false, message);
                 com.discord.api.user.User author2 = message.getAuthor();
                 if (author2 != null) {
                     l = Long.valueOf(author2.i());
