--- ./discord_94011/src/main/java/com/discord/widgets/status/WidgetGlobalStatusIndicator.java	2021-12-24 01:10:15.363025400 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/status/WidgetGlobalStatusIndicator.java	2021-12-24 01:08:35.836262700 +0400
@@ -58,6 +58,10 @@ public final class WidgetGlobalStatusInd
         this.connectingVectorReplayCallback = new WidgetGlobalStatusIndicator$connectingVectorReplayCallback$1(this);
     }
 
+    public static final /* synthetic */ AnimatedVectorDrawableCompat access$getConnectingVector$p(WidgetGlobalStatusIndicator widgetGlobalStatusIndicator) {
+        return widgetGlobalStatusIndicator.connectingVector;
+    }
+
     private final Observable<WidgetGlobalStatusIndicatorViewModel.ViewState> bindDelay(Observable<WidgetGlobalStatusIndicatorViewModel.ViewState> observable) {
         return observable.X(new WidgetGlobalStatusIndicator$bindDelay$1(observable));
     }
