--- ./discord_94011/src/main/java/com/discord/widgets/status/WidgetGlobalStatusIndicator$connectingVectorReplayCallback$1.java	2021-12-24 01:10:15.365027200 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/status/WidgetGlobalStatusIndicator$connectingVectorReplayCallback$1.java	2021-12-24 01:08:35.838261100 +0400
@@ -18,9 +18,9 @@ public final class WidgetGlobalStatusInd
     @Override // androidx.vectordrawable.graphics.drawable.Animatable2Compat.AnimationCallback
     public void onAnimationEnd(Drawable drawable) {
         m.checkNotNullParameter(drawable, "drawable");
-        AnimatedVectorDrawableCompat animatedVectorDrawableCompat = this.this$0.connectingVector;
-        if (animatedVectorDrawableCompat != null) {
-            animatedVectorDrawableCompat.start();
+        AnimatedVectorDrawableCompat access$getConnectingVector$p = WidgetGlobalStatusIndicator.access$getConnectingVector$p(this.this$0);
+        if (access$getConnectingVector$p != null) {
+            access$getConnectingVector$p.start();
         }
     }
 }
