--- ./discord_94011/src/main/java/com/discord/widgets/settings/account/mfa/WidgetEnableMFAKey$onViewBound$1.java	2021-12-24 01:10:15.154024900 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/settings/account/mfa/WidgetEnableMFAKey$onViewBound$1.java	2021-12-24 01:08:35.653262100 +0400
@@ -19,6 +19,6 @@ public final class WidgetEnableMFAKey$on
         m.checkNotNullExpressionValue(view, "it");
         Context context = view.getContext();
         m.checkNotNullExpressionValue(context, "it.context");
-        WidgetEnableMFAKey.access$copyCodeToClipboard(widgetEnableMFAKey, context);
+        widgetEnableMFAKey.copyCodeToClipboard(context);
     }
 }
