--- ./discord_94011/src/main/java/c/i/c/m/d/n/a.java	2021-12-24 01:10:11.388025300 +0400
+++ ./bluecord/src/main/java/c/i/c/m/d/n/a.java	2021-12-24 01:08:31.865264100 +0400
@@ -105,7 +105,7 @@ public class a {
     }
 
     public a c(String str, String str2, String str3, File file) {
-        MediaType.a aVar = MediaType.f2950c;
+        MediaType.a aVar = MediaType.f2952c;
         RequestBody create = RequestBody.create(MediaType.a.b(str3), file);
         if (this.f == null) {
             MultipartBody.a aVar2 = new MultipartBody.a();
