--- ./discord_94011/src/main/java/com/discord/widgets/channels/WidgetChannelTopic$onViewBoundOrOnResume$1.java	2021-12-24 01:10:13.732025500 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/channels/WidgetChannelTopic$onViewBoundOrOnResume$1.java	2021-12-24 01:08:34.116265500 +0400
@@ -28,6 +28,6 @@ public final class WidgetChannelTopic$on
 
     public final void invoke(WidgetChannelTopicViewModel.Event event) {
         m.checkNotNullParameter(event, "event");
-        WidgetChannelTopic.access$handleEvent(this.this$0, event);
+        this.this$0.handleEvent(event);
     }
 }
