--- ./discord_94011/src/main/java/com/discord/widgets/channels/WidgetChannelTopic$configureUI$2.java	2021-12-24 01:10:13.731025800 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/channels/WidgetChannelTopic$configureUI$2.java	2021-12-24 01:08:34.115264000 +0400
@@ -16,6 +16,6 @@ public final class WidgetChannelTopic$co
     }
 
     public final void onClick(View view) {
-        WidgetChannelTopic.access$onClickMore(this.this$0, this.$viewState);
+        this.this$0.onClickMore(this.$viewState);
     }
 }
