--- ./discord_94011/src/main/java/com/discord/widgets/channels/WidgetChannelTopic$onToggleTopicExpansionState$1.java	2021-12-24 01:10:13.732025500 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/channels/WidgetChannelTopic$onToggleTopicExpansionState$1.java	2021-12-24 01:08:34.115264000 +0400
@@ -27,7 +27,7 @@ public final class WidgetChannelTopic$on
     }
 
     public final void invoke(int i) {
-        ObjectAnimator ofInt = ObjectAnimator.ofInt(WidgetChannelTopic.access$getBinding$p(this.this$0).g, "maxLines", i);
+        ObjectAnimator ofInt = ObjectAnimator.ofInt(this.this$0.getBinding().g, "maxLines", i);
         m.checkNotNullExpressionValue(ofInt, "animation");
         ofInt.setDuration(150L);
         ofInt.addListener(new WidgetChannelTopic$onToggleTopicExpansionState$1$animateMaxLines$$inlined$doOnEnd$1(this));
