--- ./discord_94011/src/main/java/com/discord/widgets/channels/WidgetChannelSidebarActions$configureUI$1.java	2021-12-24 01:10:13.724025600 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/channels/WidgetChannelSidebarActions$configureUI$1.java	2021-12-24 01:08:34.107262000 +0400
@@ -16,6 +16,6 @@ public final class WidgetChannelSidebarA
     }
 
     public final void onClick(View view) {
-        this.this$0.startPrivateCall(((WidgetChannelSidebarActionsViewModel.ViewState.Private) this.$viewState).getChannelId(), false);
+        WidgetChannelSidebarActions.access$startPrivateCall(this.this$0, ((WidgetChannelSidebarActionsViewModel.ViewState.Private) this.$viewState).getChannelId(), false);
     }
 }
