--- ./discord_94011/src/main/java/com/discord/widgets/feedback/WidgetFeedbackSheet.java	2021-12-24 01:10:14.270024900 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/feedback/WidgetFeedbackSheet.java	2021-12-24 01:08:34.861263100 +0400
@@ -111,10 +111,6 @@ public final class WidgetFeedbackSheet e
         this.viewModelGuildDeleteFeedbackSheet$delegate = FragmentViewModelLazyKt.createViewModelLazy(this, a0.getOrCreateKotlinClass(GuildDeleteFeedbackSheetViewModel.class), new WidgetFeedbackSheet$appViewModels$$inlined$viewModels$3(f0Var3), new h0(widgetFeedbackSheet$viewModelGuildDeleteFeedbackSheet$2));
     }
 
-    public static final /* synthetic */ void access$updateView(WidgetFeedbackSheet widgetFeedbackSheet, FeedbackSheetViewModel.ViewState viewState) {
-        widgetFeedbackSheet.updateView(viewState);
-    }
-
     private final WidgetFeedbackSheetBinding getBinding() {
         return (WidgetFeedbackSheetBinding) this.binding$delegate.getValue((Fragment) this, $$delegatedProperties[0]);
     }
@@ -168,7 +164,8 @@ public final class WidgetFeedbackSheet e
         }
     }
 
-    private final void updateView(FeedbackSheetViewModel.ViewState viewState) {
+    /* access modifiers changed from: private */
+    public final void updateView(FeedbackSheetViewModel.ViewState viewState) {
         boolean z2 = true;
         if (this.isShowingFeedbackIssues || !(!viewState.getFeedbackIssues().isEmpty())) {
             z2 = false;
