--- ./discord_94011/src/main/java/com/discord/widgets/feedback/WidgetFeedbackSheet$bindSubscriptions$1.java	2021-12-24 01:10:14.272030000 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/feedback/WidgetFeedbackSheet$bindSubscriptions$1.java	2021-12-24 01:08:34.862264900 +0400
@@ -24,6 +24,6 @@ public final /* synthetic */ class Widge
 
     public final void invoke(FeedbackSheetViewModel.ViewState viewState) {
         m.checkNotNullParameter(viewState, "p1");
-        WidgetFeedbackSheet.access$updateView((WidgetFeedbackSheet) this.receiver, viewState);
+        ((WidgetFeedbackSheet) this.receiver).updateView(viewState);
     }
 }
