--- ./discord_94011/src/main/java/com/discord/widgets/chat/input/gifpicker/GifSearchViewModel$Companion$observeStoreState$1.java	2021-12-24 01:10:14.018026200 +0400
+++ ./bluecord/src/main/java/com/discord/widgets/chat/input/gifpicker/GifSearchViewModel$Companion$observeStoreState$1.java	2021-12-24 01:08:34.419261600 +0400
@@ -36,6 +36,6 @@ public final class GifSearchViewModel$Co
         StoreAnalytics storeAnalytics = this.$storeAnalytics;
         StoreGifPicker storeGifPicker = this.$storeGifPicker;
         m.checkNotNullExpressionValue(list, "terms");
-        return companion.observeQueryState(behaviorSubject, storeAnalytics, storeGifPicker, list);
+        return GifSearchViewModel.Companion.access$observeQueryState(companion, behaviorSubject, storeAnalytics, storeGifPicker, list);
     }
 }
